diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Lang/50/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Lang/50/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Lang/50/buggy1.java	2021-08-27 19:18:38.986864096 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Lang/50/tool-patch1.java	2021-08-27 19:18:38.986864096 -0700
@@ -472,6 +472,9 @@
                 locale = Locale.getDefault();
             }
             try {
+                if(locale!=null){
+                    key=new Pair(key,locale);
+                }
                 SimpleDateFormat formatter = (SimpleDateFormat) DateFormat.getDateTimeInstance(dateStyle, timeStyle,
                         locale);
                 String pattern = formatter.toPattern();
