diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/Jaid/Math/53/3/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/Jaid/Math/53/3/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/Jaid/Math/53/3/buggy1.java	2021-08-27 19:18:44.246864245 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/Jaid/Math/53/3/tool-patch1.java	2021-08-27 19:18:44.246864245 -0700
@@ -150,6 +150,10 @@
     public Complex add(Complex rhs)
         throws NullArgumentException {
         MathUtils.checkNotNull(rhs);
+        if((rhs.isNaN() || isInfinite()) == true){
+        	return this.NaN;
+
+        	}
         return createComplex(real + rhs.getReal(),
             imaginary + rhs.getImaginary());
     }
