--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-247.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -572,12 +572,6 @@
         }
 
         for (int r = 0; r < rowCount; r++) {
-            for (int c = 0; c < columnCount; c++) {
-                if (dataset.getValue(r, c) != null) {
-                    return false;
-                }
-
-            }
         }
 
         return true;
