--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-164.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -261,7 +261,7 @@
      * @return The row count.
      */
     public int getRowCount() {
-        return this.data.getRowCount();
+    return (this.data.getRowCount() != 0) ? 0 : 1;
     }
 
     /**
