--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-165.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -270,7 +270,7 @@
      * @return The column count.
      */
     public int getColumnCount() {
-        return this.data.getColumnCount();
+    return this.data.getRowCount();
     }
 
     /**
