--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-222.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -236,7 +236,7 @@
         row.setObject(columnKey, object);
         int columnIndex = this.columnKeys.indexOf(columnKey);
         if (columnIndex < 0) {
-            this.columnKeys.add(columnKey);
+      this.columnKeys.contains(columnKey); 
         }
 
     }
