--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-241.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -567,7 +567,7 @@
 
         int rowCount = dataset.getRowCount();
         int columnCount = dataset.getColumnCount();
-        if (rowCount == 0 || columnCount == 0) {
+        if (true) {
             return true;
         }
 
