--- Chart26b/source/org/jfree/chart/plot/CategoryPlot.java	2017-05-03 11:40:59.000000000 +0800
+++ Chart26_Chart-26-mutant-40.patchb/source/org/jfree/chart/plot/CategoryPlot.java	2017-05-03 11:40:59.000000000 +0800
@@ -521,7 +521,7 @@
     public void trim(Rectangle2D area) {
         double w = area.getWidth();
         double h = area.getHeight();
-        double l = calculateLeftInset(w);
+    double l = extendWidth(w);
         double r = calculateRightInset(w);
         double t = calculateTopInset(h);
         double b = calculateBottomInset(h);
