--- Chart26b/source/org/jfree/chart/plot/CategoryPlot.java	2017-05-03 11:40:59.000000000 +0800
+++ Chart26_Chart-26-mutant-73.patchb/source/org/jfree/chart/plot/CategoryPlot.java	2017-05-03 11:40:59.000000000 +0800
@@ -1210,7 +1210,6 @@
         if (info != null) {
             plotInfo = info.getPlotInfo();
         }
-        this.plot.draw(g2, plotArea, anchor, null, plotInfo);
 
         g2.setClip(savedClip);
 
