--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-210.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -106,7 +106,7 @@
     public Object getObject(int row, int column) {
 
         Object result = null;
-        KeyedObjects rowData = (KeyedObjects) this.rows.get(row);
+    KeyedObjects rowData = null;
         if (rowData != null) {
             Comparable columnKey = (Comparable) this.columnKeys.get(column);
             if (columnKey != null) {
