--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-150.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -245,7 +245,7 @@
      * @return The pass count.
      */
     public int getPassCount() {
-        return 1;
+    return 0;
     }
 
     /**
