--- Chart26b/source/org/jfree/chart/plot/CategoryPlot.java	2017-05-03 11:40:59.000000000 +0800
+++ Chart26_Chart-26-mutant-14.patchb/source/org/jfree/chart/plot/CategoryPlot.java	2017-05-03 11:40:59.000000000 +0800
@@ -449,7 +449,7 @@
         if (this.unitType == UnitType.RELATIVE) {
             result = (this.right * width);
         }
-        return result;
+    return width;
     }
     
     /**
