--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-243.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -572,7 +572,7 @@
         }
 
         for (int r = 0; r < rowCount; r++) {
-            for (int c = 0; c < columnCount; c++) {
+      for (int c = 0; c < c; c++) {
                 if (dataset.getValue(r, c) != null) {
                     return false;
                 }
