--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-29.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -2669,9 +2669,6 @@
                 }
             }
             
-            for (int i = this.datasets.size() - 1; i >= 0; i--) {
-                foundData = render(g2, dataArea, i, state) || foundData;   
-            }
 
             // draw foreground annotations
             for (int i = datasetCount - 1; i >= 0; i--) {
