--- Chart26b/source/org/jfree/chart/plot/CategoryPlot.java	2017-05-03 11:40:59.000000000 +0800
+++ Chart26_Chart-26-mutant-91.patchb/source/org/jfree/chart/plot/CategoryPlot.java	2017-05-03 11:40:59.000000000 +0800
@@ -339,7 +339,9 @@
      * @see #setLabel(String)
      */
     public String getLabel() {
-        return this.label;
+    if (this.label == null)
+      throw new RuntimeException(); 
+    return null;
     }
 
     /**
