--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-190.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -109,7 +109,7 @@
         KeyedObjects rowData = (KeyedObjects) this.rows.get(row);
         if (rowData != null) {
             Comparable columnKey = (Comparable) this.columnKeys.get(column);
-            if (columnKey != null) {
+      if (columnKey == null) {
                 result = rowData.getObject(columnKey);
             }
         }
