--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-218.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -92,7 +92,7 @@
      * @return The column count.
      */
     public int getColumnCount() {
-        return this.columnKeys.size();
+    return (this.columnKeys.size() != 0) ? 0 : 1;
     }
 
     /**
