--- Chart15b/source/org/jfree/chart/plot/PiePlot3D.java	2017-03-20 23:22:52.000000000 +0800
+++ Chart15_Chart-15-mutant-1.patchb/source/org/jfree/chart/plot/PiePlot3D.java	2017-03-20 23:22:52.000000000 +0800
@@ -1216,7 +1216,6 @@
         if (info != null) {
             plotInfo = info.getPlotInfo();
         }
-        this.plot.draw(g2, plotArea, anchor, null, plotInfo);
 
         g2.setClip(savedClip);
 
