--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-199.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -235,9 +235,6 @@
         }
         row.setObject(columnKey, object);
         int columnIndex = this.columnKeys.indexOf(columnKey);
-        if (columnIndex < 0) {
-            this.columnKeys.add(columnKey);
-        }
 
     }
 
