--- Chart13b/source/org/jfree/chart/block/BorderArrangement.java	2017-03-20 23:22:39.000000000 +0800
+++ Chart13_Chart-13-mutant-9.patchb/source/org/jfree/chart/block/BorderArrangement.java	2017-03-20 23:22:39.000000000 +0800
@@ -446,7 +446,9 @@
                     new Range(0.0, constraint.getWidth()),
                     LengthConstraintType.RANGE, h[2], null,
                     LengthConstraintType.FIXED);
-            Size2D size = this.leftBlock.arrange(g2, c3);
+      RectangleConstraint rectangleConstraint1 = c3;
+      Graphics2D graphics2D = g2;
+      Size2D size = new Size2D(0.0D, 0.0D);
             w[2] = size.width;
         }
         h[3] = h[2];
