--- Chart26b/source/org/jfree/chart/plot/CategoryPlot.java	2017-05-03 11:40:59.000000000 +0800
+++ Chart26_Chart-26-mutant-10.patchb/source/org/jfree/chart/plot/CategoryPlot.java	2017-05-03 11:40:59.000000000 +0800
@@ -524,7 +524,7 @@
         double l = calculateLeftInset(w);
         double r = calculateRightInset(w);
         double t = calculateTopInset(h);
-        double b = calculateBottomInset(h);
+    double b = h;
         area.setRect(area.getX() + l, area.getY() + t, w - l - r, h - t - b);    
     }
     
