--- Chart26b/source/org/jfree/chart/plot/CategoryPlot.java	2017-05-03 11:40:59.000000000 +0800
+++ Chart26_Chart-26-mutant-74.patchb/source/org/jfree/chart/plot/CategoryPlot.java	2017-05-03 11:40:59.000000000 +0800
@@ -1309,7 +1309,7 @@
             titleArea = createAlignedRectangle2D(size, area, 
                     t.getHorizontalAlignment(), VerticalAlignment.BOTTOM);
             retValue = t.draw(g2, titleArea, p);
-            area.setRect(area.getX(), area.getY(), area.getWidth(), 
+      area.setRect(area.getX(), area.getY(), 0.0D, 
                     area.getHeight() - size.height);
         }
         else if (position == RectangleEdge.RIGHT) {
