--- Chart26b/source/org/jfree/chart/plot/CategoryPlot.java	2017-05-03 11:40:59.000000000 +0800
+++ Chart26_Chart-26-mutant-57.patchb/source/org/jfree/chart/plot/CategoryPlot.java	2017-05-03 11:40:59.000000000 +0800
@@ -1310,7 +1310,7 @@
                     t.getHorizontalAlignment(), VerticalAlignment.BOTTOM);
             retValue = t.draw(g2, titleArea, p);
             area.setRect(area.getX(), area.getY(), area.getWidth(), 
-                    area.getHeight() - size.height);
+      titleArea.getHeight() - size.height);
         }
         else if (position == RectangleEdge.RIGHT) {
             Size2D size = t.arrange(g2, constraint);
