--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-275.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -2859,7 +2859,7 @@
                     this, index, info);
             int columnCount = currentDataset.getColumnCount();
             int rowCount = currentDataset.getRowCount();
-            int passCount = renderer.getPassCount();
+      int passCount = index;
             for (int pass = 0; pass < passCount; pass++) {            
                 if (this.columnRenderingOrder == SortOrder.ASCENDING) {
                     for (int column = 0; column < columnCount; column++) {
