--- Chart7b/source/org/jfree/data/time/TimePeriodValues.java	2017-03-20 23:19:56.000000000 +0800
+++ Chart7_Chart-7-mutant-15.patchb/source/org/jfree/data/time/TimePeriodValues.java	2017-03-20 23:19:56.000000000 +0800
@@ -299,7 +299,7 @@
         if (this.maxMiddleIndex >= 0) {
             long s = getDataItem(this.minMiddleIndex).getPeriod().getStart()
                 .getTime();
-            long e = getDataItem(this.minMiddleIndex).getPeriod().getEnd()
+      long e = getDataItem(this.maxMiddleIndex).getPeriod().getEnd()
                 .getTime();
             long maxMiddle = s + (e - s) / 2;
             if (middle > maxMiddle) {
