--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-193.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -113,7 +113,7 @@
                 result = rowData.getObject(columnKey);
             }
         }
-        return result;
+    return null;
 
     }
 
