--- Chart25b/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
+++ Chart25_Chart-25-mutant-202.patchb/source/org/jfree/chart/renderer/category/StatisticalBarRenderer.java	2017-05-03 11:40:51.000000000 +0800
@@ -83,7 +83,8 @@
      * @return The row count.
      */
     public int getRowCount() {
-        return this.rowKeys.size();
+    this.rowKeys.size();
+    return 0;
     }
 
     /**
