--- Lang57b/src/java/org/apache/commons/lang/LocaleUtils.java	2017-08-23 00:18:52.709143656 +0800
+++ Lang57_Lang-57-mutant-1.patchb/src/java/org/apache/commons/lang/LocaleUtils.java	2017-08-23 00:18:52.709143656 +0800
@@ -220,7 +220,8 @@
      * @return true if the locale is a known locale
      */
     public static boolean isAvailableLocale(Locale locale) {
-        return cAvailableLocaleSet.contains(locale);
+    Locale locale1 = locale;
+    return (cAvailableLocaleSet == null) ? false : cAvailableLocaleSet.contains(locale1);
     }
 
     //-----------------------------------------------------------------------
