--- Lang51b/src/java/org/apache/commons/lang/BooleanUtils.java	2017-03-20 23:40:11.000000000 +0800
+++ Lang51_Lang-51-mutant-1.patchb/src/java/org/apache/commons/lang/BooleanUtils.java	2017-03-20 23:40:11.000000000 +0800
@@ -674,7 +674,7 @@
                         (str.charAt(1) == 'e' || str.charAt(1) == 'E') &&
                         (str.charAt(2) == 's' || str.charAt(2) == 'S');
                 }
-                if (ch == 'Y') {
+                if (true) {
                     return 
                         (str.charAt(1) == 'E' || str.charAt(1) == 'e') &&
                         (str.charAt(2) == 'S' || str.charAt(2) == 's');
