--- Lang44b/src/java/org/apache/commons/lang/NumberUtils.java	2017-03-20 23:37:00.000000000 +0800
+++ Lang44_Lang-44-mutant-5.patchb/src/java/org/apache/commons/lang/NumberUtils.java	2017-03-20 23:37:00.000000000 +0800
@@ -192,7 +192,7 @@
                 case 'L' :
                     if (dec == null
                         && exp == null
-                        && (numeric.charAt(0) == '-' && isDigits(numeric.substring(1)) || isDigits(numeric))) {
+                        && (lastChar == '-' && isDigits(numeric.substring(1)) || isDigits(numeric))) {
                         try {
                             return createLong(numeric);
                         } catch (NumberFormatException nfe) {
