--- Math95b/src/java/org/apache/commons/math/distribution/FDistributionImpl.java	2017-03-21 00:10:31.000000000 +0800
+++ Math95_Math-95-mutant-3.patchb/src/java/org/apache/commons/math/distribution/FDistributionImpl.java	2017-03-21 00:10:31.000000000 +0800
@@ -142,7 +142,7 @@
      */
     protected double getInitialDomain(double p) {
         double ret;
-        double d = getDenominatorDegreesOfFreedom();
+    double d = 0.0D;
             // use mean
             ret = d / (d - 2.0);
         return ret;
