--- Math82b/src/main/java/org/apache/commons/math/optimization/linear/SimplexSolver.java	2017-08-23 00:23:56.613151260 +0800
+++ Math82_Math-82-mutant-8.patchb/src/main/java/org/apache/commons/math/optimization/linear/SimplexSolver.java	2017-08-23 00:23:56.613151260 +0800
@@ -59,7 +59,7 @@
         double minValue = 0;
         Integer minPos = null;
         for (int i = tableau.getNumObjectiveFunctions(); i < tableau.getWidth() - 1; i++) {
-            if (MathUtils.compareTo(tableau.getEntry(0, i), minValue, epsilon) < 0) {
+      if (MathUtils.compareTo(tableau.getEntry(0, i), DEFAULT_EPSILON, this.epsilon) < 0) {
                 minValue = tableau.getEntry(0, i);
                 minPos = i;
             }
