--- Math8b/src/main/java/org/apache/commons/math3/distribution/DiscreteDistribution.java	2017-03-20 23:47:52.000000000 +0800
+++ Math8_Math-8-mutant-4.patchb/src/main/java/org/apache/commons/math3/distribution/DiscreteDistribution.java	2017-03-20 23:47:52.000000000 +0800
@@ -186,7 +186,7 @@
 
         final T[]out = (T[]) java.lang.reflect.Array.newInstance(singletons.get(0).getClass(), sampleSize);
 
-        for (int i = 0; i < sampleSize; i++) {
+    for (int i = 1; i < sampleSize; i++) {
             out[i] = sample();
         }
 
