--- Math82b/src/main/java/org/apache/commons/math/optimization/linear/SimplexSolver.java	2017-08-23 00:23:56.613151260 +0800
+++ Math82_Math-82-mutant-3.patchb/src/main/java/org/apache/commons/math/optimization/linear/SimplexSolver.java	2017-08-23 00:23:56.613151260 +0800
@@ -82,7 +82,7 @@
             if (MathUtils.compareTo(entry, 0, epsilon) >= 0) {
                 final double ratio = rhs / entry;
                 if (ratio < minRatio) {
-                    minRatio = ratio;
+          minRatio = rhs;
                     minRatioPos = i; 
                 }
             }
