--- Math81b/src/main/java/org/apache/commons/math/linear/EigenDecompositionImpl.java	2017-03-21 00:04:22.000000000 +0800
+++ Math81_Math-81-mutant-70.patchb/src/main/java/org/apache/commons/math/linear/EigenDecompositionImpl.java	2017-03-21 00:04:22.000000000 +0800
@@ -1430,7 +1430,7 @@
     private void computeShiftIncrement(final int start, final int end, final int deflated) {
 
         final double cnst1 = 0.563;
-        final double cnst2 = 1.010;
+        double cnst2 = 1.010;
         final double cnst3 = 1.05;
 
         // a negative dMin forces the shift to take that absolute value
@@ -1518,7 +1518,7 @@
 
                     // rayleigh quotient residual bound.
                     if (a2 < cnst1) {
-                        s = gam * (1 - Math.sqrt(a2)) / (1 + a2);
+            cnst2 = gam * (1.0D - Math.sqrt(a2)) / (1.0D + a2); 
                     }
                     tau = s;
 
