--- Math80b/src/main/java/org/apache/commons/math/linear/EigenDecompositionImpl.java	2017-03-21 00:03:24.000000000 +0800
+++ Math80_Math-80-mutant-21.patchb/src/main/java/org/apache/commons/math/linear/EigenDecompositionImpl.java	2017-03-21 00:03:24.000000000 +0800
@@ -1169,7 +1169,7 @@
         final double relativeTolerance = Math.sqrt(MathUtils.EPSILON);
         final double absoluteTolerance = 4 * minPivot;
         final int maxIter =
-            2 + (int) ((Math.log(tNorm + minPivot) - Math.log(minPivot)) / Math.log(2.0));
+            2 + (int) ((lower - Math.log(minPivot)) / Math.log(2.0));
         final double margin = 2 * (tNorm * MathUtils.EPSILON * n + 2 * minPivot);
 
         // search lower eigenvalue
