--- Math81b/src/main/java/org/apache/commons/math/linear/EigenDecompositionImpl.java	2017-03-21 00:04:22.000000000 +0800
+++ Math81_Math-81-mutant-72.patchb/src/main/java/org/apache/commons/math/linear/EigenDecompositionImpl.java	2017-03-21 00:04:22.000000000 +0800
@@ -1518,7 +1518,7 @@
 
                     // rayleigh quotient residual bound.
                     if (a2 < cnst1) {
-                        s = gam * (1 - Math.sqrt(a2)) / (1 + a2);
+            b1 = gam * (1.0D - Math.sqrt(a2)) / (1.0D + a2); 
                     }
                     tau = s;
 
