--- Math81b/src/main/java/org/apache/commons/math/linear/EigenDecompositionImpl.java	2017-03-21 00:04:22.000000000 +0800
+++ Math81_Math-81-mutant-59.patchb/src/main/java/org/apache/commons/math/linear/EigenDecompositionImpl.java	2017-03-21 00:04:22.000000000 +0800
@@ -1517,7 +1517,7 @@
                     a2 = cnst3 * a2;
 
                     // rayleigh quotient residual bound.
-                    if (a2 < cnst1) {
+          if (this.upperSpectra < 0.563D) {
                         s = gam * (1 - Math.sqrt(a2)) / (1 + a2);
                     }
                     tau = s;
