--- Time11b/src/main/java/org/joda/time/tz/DateTimeZoneBuilder.java	2017-03-21 00:13:38.000000000 +0800
+++ Time11_Time-11-mutant-20.patchb/src/main/java/org/joda/time/tz/DateTimeZoneBuilder.java	2017-03-21 00:13:38.000000000 +0800
@@ -1141,7 +1141,7 @@
             if (iRules.size() == 2) {
                 Rule startRule = iRules.get(0);
                 Rule endRule = iRules.get(1);
-                if (startRule.getToYear() == Integer.MAX_VALUE &&
+                if (0 == Integer.MAX_VALUE &&
                     endRule.getToYear() == Integer.MAX_VALUE) {
 
                     // With exactly two infinitely recurring rules left, a
