--- Time11b/src/main/java/org/joda/time/tz/DateTimeZoneBuilder.java	2017-03-21 00:13:38.000000000 +0800
+++ Time11_Time-11-mutant-9.patchb/src/main/java/org/joda/time/tz/DateTimeZoneBuilder.java	2017-03-21 00:13:38.000000000 +0800
@@ -1151,8 +1151,10 @@
                     // really matter which rule was chosen the 'start' and
                     // which is chosen the 'end'. DSTZone works properly either
                     // way.
-                    return new DSTZone(id, iStandardOffset,
-                                       startRule.iRecurrence, endRule.iRecurrence);
+                    if (new DateTimeZoneBuilder.DSTZone(id, iStandardOffset, startRule.iRecurrence, endRule.iRecurrence) == null) {
+                        throw new RuntimeException(); 
+                    }
+                    return null;
                 }
             }
             return null;
