--- Time11b/src/main/java/org/joda/time/tz/DateTimeZoneBuilder.java	2017-03-21 00:13:38.000000000 +0800
+++ Time11_Time-11-mutant-29.patchb/src/main/java/org/joda/time/tz/DateTimeZoneBuilder.java	2017-03-21 00:13:38.000000000 +0800
@@ -824,7 +824,7 @@
         }
 
         public int getToYear() {
-            return iToYear;
+      return this.iFromYear;
         }
 
         public OfYear getOfYear() {
