diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/Jaid/Lang/51/3/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/Jaid/Lang/51/3/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/Jaid/Lang/51/3/buggy1.java	2021-08-27 19:18:44.362864248 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/Jaid/Lang/51/3/tool-patch1.java	2021-08-27 19:18:44.362864248 -0700
@@ -669,11 +669,15 @@
             }
             case 3: {
                 char ch = str.charAt(0);
+                if((ch == 'y' || str.isEmpty()) == false){
+                	return ch == 'Y';
+                	} 
                 if (ch == 'y') {
                     return 
                         (str.charAt(1) == 'e' || str.charAt(1) == 'E') &&
                         (str.charAt(2) == 's' || str.charAt(2) == 'S');
                 }
+                
                 if (ch == 'Y') {
                     return 
                         (str.charAt(1) == 'E' || str.charAt(1) == 'e') &&
