diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Arja/Math/40/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Arja/Math/40/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Arja/Math/40/buggy1.java	2021-08-27 19:18:32.786863921 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Arja/Math/40/tool-patch1.java	2021-08-27 19:18:32.786863921 -0700
@@ -261,6 +261,7 @@
                         // we have more points before the sign change, drop the lowest point
                         ++start;
                     } else {
+                        ++start;
                         // we have more points after sign change, drop the highest point
                         --end;
                     }
