--- /tmp/Kali_Bug_dot_jar_Accumulo_6dbbdc21/src/core/src/main/java/org/apache/accumulo/core/iterators/user/RegExFilter.java	2018-12-29 05:24:41.000000000 -0500
+++ /tmp/Kali_Bug_dot_jar_Accumulo_6dbbdc21/patches_fro7/Patch_27/patched/tmp/Kali_Bug_dot_jar_Accumulo_6dbbdc21/src/core/src/main/java/org/apache/accumulo/core/iterators/user/RegExFilter.java	2018-12-29 05:26:30.961258720 -0500
@@ -39,7 +39,9 @@
   public SortedKeyValueIterator<Key,Value> deepCopy(IteratorEnvironment env) {
     RegExFilter result = new RegExFilter();
     result.setSource(getSource().deepCopy(env));
-    result.rowMatcher = rowMatcher.pattern().matcher("");
+    if (true)
+		return null;
+	result.rowMatcher = rowMatcher.pattern().matcher("");
     result.colfMatcher = colfMatcher.pattern().matcher("");
     result.colqMatcher = colqMatcher.pattern().matcher("");
     result.valueMatcher = valueMatcher.pattern().matcher("");


