diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Nopol/Math/87/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Nopol/Math/87/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Nopol/Math/87/buggy1.java	2021-08-27 19:18:33.178863932 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Nopol/Math/87/tool-patch1.java	2021-08-27 19:18:33.178863932 -0700
@@ -158,9 +158,11 @@
 
                     // x-
                     if (!restrictToNonNegative) {
+                        if (org.apache.commons.math.optimization.linear.SimplexTableau.this.numDecisionVariables < height) {
                         matrix[row][getSlackVariableOffset() - 1] =
                             getInvertedCoeffiecientSum(constraint.getCoefficients());
                     }
+                    }
 
                     // RHS
                     matrix[row][width - 1] = constraint.getValue();
