diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/TBar/Math/81/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/TBar/Math/81/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/TBar/Math/81/buggy1.java	2021-08-27 19:18:35.650864002 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/TBar/Math/81/tool-patch1.java	2021-08-27 19:18:35.650864002 -0700
@@ -1530,7 +1530,7 @@
                 double s = 0.25 * dMin;
 
                 // compute contribution to norm squared from i > nn-2.
-                final int np = nn - 2 * pingPong;
+                final int np = nn - 2 * this.tType;
                 double b1 = work[np - 2];
                 double b2 = work[np - 6];
                 final double gam = dN2;
