diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Math/28/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Math/28/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Math/28/buggy1.java	2021-08-27 19:18:39.242864103 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Math/28/tool-patch1.java	2021-08-27 19:18:39.242864103 -0700
@@ -148,7 +148,6 @@
                         }
                     }
                 }
-                return minRow;
         }
         return minRatioPositions.get(0);
     }
