diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Closure/46/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Closure/46/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Closure/46/buggy1.java	2021-08-27 19:18:38.978864096 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Closure/46/tool-patch1.java	2021-08-27 19:18:38.978864096 -0700
@@ -151,7 +151,7 @@
             getPropertyNode(property));
       }
     }
-    return builder.build();
+    return getLeastSupertype(this,that);
   }
   JSType getGreatestSubtypeHelper(JSType that) {
     if (that.isRecordType()) {
