diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/4/mutant-0/buggy-XYPlot.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/4/mutant-0/patched-XYPlot.java
--- Chart4b/source/org/jfree/chart/plot/XYPlot.java
+++ Chart4b_Chart-4-src.patchb/source/org/jfree/chart/plot/XYPlot.java
@@ -4490,6 +4490,7 @@
                     }
                 }
                 
+                if (r != null) {
                     Collection c = r.getAnnotations();
                     Iterator i = c.iterator();
                     while (i.hasNext()) {
@@ -4500,6 +4501,7 @@
                     }
             }
         }
+        }
 
         Iterator it = includedAnnotations.iterator();
         while (it.hasNext()) {
