diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/11/mutant-0/buggy-RandomStringUtils.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/11/mutant-0/patched-RandomStringUtils.java
--- Lang11b/src/main/java/org/apache/commons/lang3/RandomStringUtils.java
+++ Lang11b_Lang-11-src.patchb/src/main/java/org/apache/commons/lang3/RandomStringUtils.java
@@ -242,6 +242,10 @@
                     start = ' ';                
                 }
             }
+        } else {
+            if (end <= start) {
+                throw new IllegalArgumentException("Parameter end (" + end + ") must be greater than start (" + start + ")");
+            }
         }
 
         char[] buffer = new char[count];
