diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/60/mutant-0/buggy-StrBuilder.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/60/mutant-0/patched-StrBuilder.java
--- Lang60b/src/java/org/apache/commons/lang/text/StrBuilder.java
+++ Lang60b_Lang-60-src.patchb/src/java/org/apache/commons/lang/text/StrBuilder.java
@@ -1670,7 +1670,7 @@
      */
     public boolean contains(char ch) {
         char[] thisBuf = buffer;
-        for (int i = 0; i < thisBuf.length; i++) {
+        for (int i = 0; i < this.size; i++) {
             if (thisBuf[i] == ch) {
                 return true;
             }
@@ -1727,7 +1727,7 @@
             return -1;
         }
         char[] thisBuf = buffer;
-        for (int i = startIndex; i < thisBuf.length; i++) {
+        for (int i = startIndex; i < size; i++) {
             if (thisBuf[i] == ch) {
                 return i;
             }
