diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/35/mutant-0/buggy-ArrayUtils.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/35/mutant-0/patched-ArrayUtils.java
--- Lang35b/src/main/java/org/apache/commons/lang3/ArrayUtils.java
+++ Lang35b_Lang-35-src.patchb/src/main/java/org/apache/commons/lang3/ArrayUtils.java
@@ -3292,7 +3292,7 @@
         } else if (element != null) {
             type = element.getClass();
         } else {
-            type = Object.class;
+            throw new IllegalArgumentException("Arguments cannot both be null");            
         }
         @SuppressWarnings("unchecked") // type must be T
         T[] newArray = (T[]) copyArrayGrow1(array, type);
@@ -3571,7 +3571,7 @@
         } else if (element != null) {
             clss = element.getClass();
         } else {
-            return (T[]) new Object[] { null };
+            throw new IllegalArgumentException("Array and element cannot both be null");            
         }
         @SuppressWarnings("unchecked") // the add method creates an array of type clss, which is type T
         final T[] newArray = (T[]) add(array, index, element, clss);
