diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/55/mutant-0/buggy-StopWatch.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/55/mutant-0/patched-StopWatch.java
--- Lang55b/src/java/org/apache/commons/lang/time/StopWatch.java
+++ Lang55b_Lang-55-src.patchb/src/java/org/apache/commons/lang/time/StopWatch.java
@@ -115,7 +115,9 @@
         if(this.runningState != STATE_RUNNING && this.runningState != STATE_SUSPENDED) {
             throw new IllegalStateException("Stopwatch is not running. ");
         }
+        if(this.runningState == STATE_RUNNING) {
             stopTime = System.currentTimeMillis();
+        }
         this.runningState = STATE_STOPPED;
     }
 
