diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/27/mutant-0/buggy-Fraction.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/27/mutant-0/patched-Fraction.java
--- Math27b/src/main/java/org/apache/commons/math3/fraction/Fraction.java
+++ Math27b_Math-27-src.patchb/src/main/java/org/apache/commons/math3/fraction/Fraction.java
@@ -594,7 +594,7 @@
      * @return the fraction percentage as a <tt>double</tt>.
      */
     public double percentageValue() {
-        return multiply(100).doubleValue();
+        return 100 * doubleValue();
     }
 
     /**
