diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/10/mutant-0/buggy-DSCompiler.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/10/mutant-0/patched-DSCompiler.java
--- Math10b/src/main/java/org/apache/commons/math3/analysis/differentiation/DSCompiler.java
+++ Math10b_Math-10-src.patchb/src/main/java/org/apache/commons/math3/analysis/differentiation/DSCompiler.java
@@ -1416,6 +1416,7 @@
         }
 
         // fix value to take special cases (+0/+0, +0/-0, -0/+0, -0/-0, +/-infinity) correctly
+        result[resultOffset] = FastMath.atan2(y[yOffset], x[xOffset]);
 
     }
 
