diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/101/mutant-0/buggy-ComplexFormat.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/101/mutant-0/patched-ComplexFormat.java
--- Math101b/src/java/org/apache/commons/math/complex/ComplexFormat.java
+++ Math101b_Math-101-src.patchb/src/java/org/apache/commons/math/complex/ComplexFormat.java
@@ -374,7 +374,8 @@
         int n = getImaginaryCharacter().length();
         startIndex = pos.getIndex();
         int endIndex = startIndex + n;
-        if (
+        if ((startIndex >= source.length()) ||
+            (endIndex > source.length()) ||
             source.substring(startIndex, endIndex).compareTo(
             getImaginaryCharacter()) != 0) {
             // set index back to initial, error index should be the start index
