diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/22/mutant-0/buggy-FDistribution.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/22/mutant-0/patched-FDistribution.java
--- Math22b/src/main/java/org/apache/commons/math3/distribution/UniformRealDistribution.java
+++ Math22b_Math-22-src.patchb/src/main/java/org/apache/commons/math3/distribution/UniformRealDistribution.java
@@ -272,7 +272,7 @@
 
     /** {@inheritDoc} */
     public boolean isSupportLowerBoundInclusive() {
-        return true;
+        return false;
     }
 
     /** {@inheritDoc} */
diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/22/mutant-0/buggy-UniformRealDistribution.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/22/mutant-0/patched-UniformRealDistribution.java
--- Math22b/src/main/java/org/apache/commons/math3/distribution/UniformRealDistribution.java
+++ Math22b_Math-22-src.patchb/src/main/java/org/apache/commons/math3/distribution/UniformRealDistribution.java
@@ -181,7 +181,7 @@
 
     /** {@inheritDoc} */
     public boolean isSupportUpperBoundInclusive() {
-        return false;
+        return true;
     }
 
     /**
