diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/69/mutant-0/buggy-PearsonsCorrelation.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/69/mutant-0/patched-PearsonsCorrelation.java
--- Math69b/src/main/java/org/apache/commons/math/stat/correlation/PearsonsCorrelation.java
+++ Math69b_Math-69-src.patchb/src/main/java/org/apache/commons/math/stat/correlation/PearsonsCorrelation.java
@@ -168,7 +168,7 @@
                 } else {
                     double r = correlationMatrix.getEntry(i, j);
                     double t = Math.abs(r * Math.sqrt((nObs - 2)/(1 - r * r)));
-                    out[i][j] = 2 * (1 - tDistribution.cumulativeProbability(t));
+                    out[i][j] = 2 * tDistribution.cumulativeProbability(-t);
                 }
             }
         }
