diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/13/mutant-0/buggy-AbstractLeastSquaresOptimizer.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/13/mutant-0/patched-AbstractLeastSquaresOptimizer.java
--- Math13b/src/main/java/org/apache/commons/math3/optimization/general/AbstractLeastSquaresOptimizer.java
+++ Math13b_Math-13-src.patchb/src/main/java/org/apache/commons/math3/optimization/general/AbstractLeastSquaresOptimizer.java
@@ -559,7 +559,16 @@
      * @return the square-root of the weight matrix.
      */
     private RealMatrix squareRoot(RealMatrix m) {
+        if (m instanceof DiagonalMatrix) {
+            final int dim = m.getRowDimension();
+            final RealMatrix sqrtM = new DiagonalMatrix(dim);
+            for (int i = 0; i < dim; i++) {
+               sqrtM.setEntry(i, i, FastMath.sqrt(m.getEntry(i, i)));
+            }
+            return sqrtM;
+        } else {
             final EigenDecomposition dec = new EigenDecomposition(m);
             return dec.getSquareRoot();
     }
+    }
 }
