diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/33/mutant-0/buggy-SimplexTableau.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/33/mutant-0/patched-SimplexTableau.java
--- Math33b/src/main/java/org/apache/commons/math3/optimization/linear/SimplexTableau.java
+++ Math33b_Math-33-src.patchb/src/main/java/org/apache/commons/math3/optimization/linear/SimplexTableau.java
@@ -335,7 +335,7 @@
         // positive cost non-artificial variables
         for (int i = getNumObjectiveFunctions(); i < getArtificialVariableOffset(); i++) {
             final double entry = tableau.getEntry(0, i);
-            if (Precision.compareTo(entry, 0d, maxUlps) > 0) {
+            if (Precision.compareTo(entry, 0d, epsilon) > 0) {
                 columnsToDrop.add(i);
             }
         }
