diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/72/mutant-0/buggy-BrentSolver.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/72/mutant-0/patched-BrentSolver.java
--- Math72b/src/main/java/org/apache/commons/math/analysis/solvers/BrentSolver.java
+++ Math72b_Math-72-src.patchb/src/main/java/org/apache/commons/math/analysis/solvers/BrentSolver.java
@@ -112,7 +112,7 @@
         // return the first endpoint if it is good enough
         double yMin = f.value(min);
         if (Math.abs(yMin) <= functionValueAccuracy) {
-            setResult(yMin, 0);
+            setResult(min, 0);
             return result;
         }
 
@@ -124,7 +124,7 @@
         // return the second endpoint if it is good enough
         double yMax = f.value(max);
         if (Math.abs(yMax) <= functionValueAccuracy) {
-            setResult(yMax, 0);
+            setResult(max, 0);
             return result;
         }
 
