diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/16/mutant-0/buggy-DateTimeFormatter.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/16/mutant-0/patched-DateTimeFormatter.java
--- Time16b/src/main/java/org/joda/time/format/DateTimeFormatter.java
+++ Time16b_Time-16-src.patchb/src/main/java/org/joda/time/format/DateTimeFormatter.java
@@ -706,7 +706,7 @@
         chrono = selectChronology(chrono);
         
         DateTimeParserBucket bucket = new DateTimeParserBucket(
-            instantLocal, chrono, iLocale, iPivotYear, iDefaultYear);
+            instantLocal, chrono, iLocale, iPivotYear, chrono.year().get(instantLocal));
         int newPos = parser.parseInto(bucket, text, position);
         instant.setMillis(bucket.computeMillis(false, text));
         if (iOffsetParsed && bucket.getOffsetInteger() != null) {
