diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/15/mutant-0/buggy-FieldUtils.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/15/mutant-0/patched-FieldUtils.java
--- Time15b/src/main/java/org/joda/time/field/FieldUtils.java
+++ Time15b_Time-15-src.patchb/src/main/java/org/joda/time/field/FieldUtils.java
@@ -135,6 +135,9 @@
     public static long safeMultiply(long val1, int val2) {
         switch (val2) {
             case -1:
+                if (val1 == Long.MIN_VALUE) {
+                    throw new ArithmeticException("Multiplication overflows a long: " + val1 + " * " + val2);
+                }
                 return -val1;
             case 0:
                 return 0L;
