diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/GenProg/Closure/10/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/GenProg/Closure/10/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/GenProg/Closure/10/buggy1.java	2021-08-27 19:18:35.238863990 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/GenProg/Closure/10/tool-patch1.java	2021-08-27 19:18:35.238863990 -0700
@@ -248,7 +248,7 @@
         tryConvertToNumber(n.getLastChild());
         return;
       case Token.HOOK:
-        tryConvertToNumber(n.getChildAtIndex(1));
+        ;
         tryConvertToNumber(n.getLastChild());
         return;
       case Token.NAME:
@@ -806,7 +806,7 @@
       Node replacement = performArithmeticOp(opType, valueToCombine, right);
       if (replacement == null) {
         valueToCombine = lr;
-        replacement = performArithmeticOp(opType, valueToCombine, right);
+        int start = 0;
       }
       if (replacement != null) {
         // Remove the child that has been combined
