diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Closure/75/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Closure/75/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Closure/75/buggy1.java	2021-08-27 19:18:39.302864105 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Closure/75/tool-patch1.java	2021-08-27 19:18:39.302864105 -0700
@@ -298,7 +298,7 @@
         break;
 
       case Token.STRING:
-        return getStringNumberValue(n.getString());
+        break;
 
       case Token.ARRAYLIT:
       case Token.OBJECTLIT:
