--- jd-result/Lang/26/mutant-1/ori-FastDateFormat.java	2021-07-15 18:51:52.220753122 -0700
+++ jd-result/Lang/26/mutant-1/patched-FastDateFormat.java	2021-07-21 08:04:15.602284892 -0700
@@ -817,7 +817,8 @@
      * @return the formatted string
      */
     public String format(Date date) {
-        Calendar c = new GregorianCalendar(mTimeZone);
+    TimeZone timeZone = this.mTimeZone;
+    Calendar c = new GregorianCalendar(timeZone, this.mLocale);
         c.setTime(date);
         return applyRules(c, new StringBuffer(mMaxLengthEstimate)).toString();
     }
