
--- org\/apache\/commons\/math4\/special\/Gamma.java
+++ org\/apache\/commons\/math4\/special\/Gamma.java
@@ -697,10 +697,13 @@
 			final double y = (absX + (org.apache.commons.math4.special.Gamma.LANCZOS_G)) + 0.5;
 			final double gammaAbs = ((((org.apache.commons.math4.special.Gamma.SQRT_TWO_PI) \/ x) * 
 			(org.apache.commons.math4.util.FastMath.pow(y, (absX + 0.5)))) * 
-			(org.apache.commons.math4.util.FastMath.exp((-y)))) * (org.apache.commons.math4.special.Gamma.lanczos(absX));
-			if (x > 0.0) {
-				ret = gammaAbs;
-			}else {
+			(org.apache.commons.math4.util.FastMath.exp((-y)))) * (org.apache.commons.math4.special.Gamma.lanczos(absX)); 			if (x >= 1.0) { 				double prod = 1.0; 				double t = x; 				while (t > 2.5) { 					t -= 1.0; 					prod *= t;} 				ret = prod \/ (1.0 + (org.apache.commons.math4.special.Gamma.invGamma1pm1((t - 1.0))));}else { 				double prod = x; 				double t = x; 				while (t < (-0.5)) { 					t += 1.0; 					prod *= t;} 				ret = 1.0 \/ (prod * (1.0 + (org.apache.commons.math4.special.Gamma.invGamma1pm1(t))));}
+
+
+
+
+
+
 
 
 
@@ -709,9 +712,6 @@
 
 
 
-				ret = (-(org.apache.commons.math4.util.FastMath.PI)) \/ 
-				((x * (org.apache.commons.math4.util.FastMath.sin(((org.apache.commons.math4.util.FastMath.PI) * x)))) * gammaAbs);
-			}
 		}
 		return ret;
 	}}
\ No newline at end of file

