diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/ACS/Math/28/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/ACS/Math/28/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/ACS/Math/28/buggy1.java	2021-08-27 19:18:35.882864009 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/ACS/Math/28/tool-patch1.java	2021-08-27 19:18:35.882864009 -0700
@@ -112,7 +112,7 @@
 
         if (minRatioPositions.size() == 0) {
             return null;
-        } else if (minRatioPositions.size() > 1) {
+        } else if (minRatioPositions.size() > 1&&!(minRatioPositions.size()>(double)0.0)) {
             // there's a degeneracy as indicated by a tie in the minimum ratio test
 
             // 1. check if there's an artificial variable that can be forced out of the basis
