diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/Jaid/Chart/26/2/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/Jaid/Chart/26/2/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/Jaid/Chart/26/2/buggy1.java	2021-08-27 19:18:44.242864244 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/Jaid/Chart/26/2/tool-patch1.java	2021-08-27 19:18:44.242864244 -0700
@@ -1187,7 +1187,7 @@
                     + insets.getRight());
 
         }
-        if (plotState != null && hotspot != null) {
+        if ((hotspot != null && plotState != null) != true && plotState != null && hotspot != null) {
             ChartRenderingInfo owner = plotState.getOwner();
                 EntityCollection entities = owner.getEntityCollection();
                 if (entities != null) {
