diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/jMutRepair/Math/57/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/jMutRepair/Math/57/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/jMutRepair/Math/57/buggy1.java	2021-08-27 19:18:34.022863956 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/jMutRepair/Math/57/tool-patch1.java	2021-08-27 19:18:34.022863956 -0700
@@ -82,7 +82,7 @@
             return false;
         }
         for (int i = 0; i < point.length; i++) {
-            if (point[i] != otherPoint[i]) {
+            if (this.point[i] != otherPoint[i]) {
                 return false;
             }
         }
