diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/kPAR/Math/40/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/kPAR/Math/40/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/kPAR/Math/40/buggy1.java	2021-08-27 19:18:35.978864011 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/kPAR/Math/40/tool-patch1.java	2021-08-27 19:18:35.978864011 -0700
@@ -257,7 +257,7 @@
                     // the guessed root is either not strictly inside the interval or it
                     // is a NaN (which occurs when some sampling points share the same y)
                     // we try again with a lower interpolation order
-                    if (signChangeIndex - start >= end - signChangeIndex) {
+                    if ((signChangeIndex - start >= end - signChangeIndex) || (y[0] * y[1] < 0)) {
                         // we have more points before the sign change, drop the lowest point
                         ++start;
                     } else {
