--- /tmp/Kali_Bears_spring-projects-spring-data-commons_185852074-193313389/src/main/java/org/springframework/data/auditing/IsNewAwareAuditingHandler.java	2018-12-27 05:27:40.459907048 -0500
+++ /tmp/Kali_Bears_spring-projects-spring-data-commons_185852074-193313389/patches_9tau/Patch_3/patched/tmp/Kali_Bears_spring-projects-spring-data-commons_185852074-193313389/src/main/java/org/springframework/data/auditing/IsNewAwareAuditingHandler.java	2018-12-27 05:28:40.303908413 -0500
@@ -73,7 +73,7 @@
 	 */
 	public void markAudited(Object object) {
 
-		if (object == null) {
+		if (true) {
 			return;
 		}
 


