diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/TBar/Closure/133/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/TBar/Closure/133/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/TBar/Closure/133/buggy1.java	2021-08-27 19:18:35.650864002 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/TBar/Closure/133/tool-patch1.java	2021-08-27 19:18:35.650864002 -0700
@@ -956,7 +956,7 @@
                         // Find the return's description (if applicable).
                         if (jsdocBuilder.shouldParseDocumentation()) {
                           ExtractionInfo returnDescriptionInfo =
-                              extractMultilineTextualBlock(token);
+                              extractMultilineTextualBlock(current());
 
                           String returnDescription =
                               returnDescriptionInfo.string;
