diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/TBar/Lang/44/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/TBar/Lang/44/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/TBar/Lang/44/buggy1.java	2021-08-27 19:18:35.814864007 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/TBar/Lang/44/tool-patch1.java	2021-08-27 19:18:35.814864007 -0700
@@ -190,9 +190,9 @@
             switch (lastChar) {
                 case 'l' :
                 case 'L' :
-                    if (dec == null
+                    if (dec != numeric || (dec == null
                         && exp == null
-                        && (numeric.charAt(0) == '-' && isDigits(numeric.substring(1)) || isDigits(numeric))) {
+                        && (numeric.charAt(0) == '-' && isDigits(numeric.substring(1)) || isDigits(numeric)))) {
                         try {
                             return createLong(numeric);
                         } catch (NumberFormatException nfe) {
