diff -u -b /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/jGenProg/Chart/7/buggy1.java /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/jGenProg/Chart/7/tool-patch1.java
--- /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/jGenProg/Chart/7/buggy1.java	2021-09-08 07:11:31.469222357 -0700
+++ /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/jGenProg/Chart/7/tool-patch1.java	2021-09-08 07:11:31.469222357 -0700
@@ -549,7 +549,7 @@
      * @return The index.
      */
     public int getMaxMiddleIndex() {
-        return this.maxMiddleIndex;
+        return this.maxEndIndex;
     }
 
     /**
