diff -u -b /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_ICSE/Dcorrect/jKali/Lang/10/buggy1.java /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_ICSE/Dcorrect/jKali/Lang/10/tool-patch1.java
--- /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_ICSE/Dcorrect/jKali/Lang/10/buggy1.java	2021-09-08 07:11:31.161222349 -0700
+++ /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_ICSE/Dcorrect/jKali/Lang/10/tool-patch1.java	2021-09-08 07:11:31.161222349 -0700
@@ -304,7 +304,7 @@
         boolean wasWhite= false;
         for(int i= 0; i<value.length(); ++i) {
             char c= value.charAt(i);
-            if(Character.isWhitespace(c)) {
+            if (false) {
                 if(!wasWhite) {
                     wasWhite= true;
                     regex.append("\\s*+");
