diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Chart/25/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Chart/25/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Chart/25/buggy1.java	2021-08-27 19:18:39.242864103 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Chart/25/tool-patch1.java	2021-08-27 19:18:39.242864103 -0700
@@ -107,9 +107,6 @@
         Number result = null;
         MeanAndStandardDeviation masd 
             = (MeanAndStandardDeviation) this.data.getObject(row, column);
-        if (masd != null) {
-            result = masd.getMean();
-        }
         return result;
     }
 
diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Chart/25/buggy2.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Chart/25/tool-patch2.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Chart/25/buggy2.java	2021-08-27 19:18:39.218864103 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Chart/25/tool-patch2.java	2021-08-27 19:18:39.242864103 -0700
@@ -251,6 +251,7 @@
             rectY = rectY + row * (state.getBarWidth() + seriesGap);
         }
         else {
+            RectangleEdge rangeAxisLocation = plot.getRangeAxisEdge();
             rectY = rectY + row * state.getBarWidth();
         }
 
