diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Chart/3/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Chart/3/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Chart/3/buggy1.java	2021-08-27 19:18:38.922864094 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Chart/3/tool-patch1.java	2021-08-27 19:18:38.922864094 -0700
@@ -1053,6 +1053,7 @@
         if (end < start) {
             throw new IllegalArgumentException("Requires start <= end.");
         }
+        this.maxY=1.0;
         TimeSeries copy = (TimeSeries) super.clone();
         copy.data = new java.util.ArrayList();
         if (this.data.size() > 0) {
