diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Dcorrect/Jaid/Chart/26/1/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Dcorrect/Jaid/Chart/26/1/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Dcorrect/Jaid/Chart/26/1/buggy1.java	2021-08-27 19:18:45.382864277 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Dcorrect/Jaid/Chart/26/1/tool-patch1.java	2021-08-27 19:18:45.382864277 -0700
@@ -1187,7 +1187,7 @@
                     + insets.getRight());
 
         }
-        if (plotState != null && hotspot != null) {
+        if (plotState != null && hotspot != null && (null == plotState.getOwner()) != true) {
             ChartRenderingInfo owner = plotState.getOwner();
                 EntityCollection entities = owner.getEntityCollection();
                 if (entities != null) {
