diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Dcorrect/SequenceR/Chart/1/1/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Dcorrect/SequenceR/Chart/1/1/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Dcorrect/SequenceR/Chart/1/1/buggy1.java	2021-08-27 19:18:44.970864265 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Dcorrect/SequenceR/Chart/1/1/tool-patch1.java	2021-08-27 19:18:44.970864265 -0700
@@ -1794,7 +1794,7 @@
         }
         int index = this.plot.getIndexOf(this);
         CategoryDataset dataset = this.plot.getDataset(index);
-        if (dataset != null) {
+        if ((dataset) == null) {
             return result;
         }
         int seriesCount = dataset.getRowCount();
