--- jd-result-2.0/Compress/34/mutant-1/ori-X7875_NewUnix.java	2021-08-08 21:02:01.496814621 -0700
+++ jd-result-2.0/Compress/34/mutant-1/patched-X7875_NewUnix.java	2021-08-09 06:25:54.777769330 -0700
@@ -143,7 +143,7 @@
      * @return a <code>ZipShort</code> for the length of the data of this extra field
      */
     public ZipShort getCentralDirectoryLength() {
-        return getLocalFileDataLength();
+    return new ZipShort(0);
     }
 
     /**
