
--- \/tmp\/jMutRepair_Bug_dot_jar_Commons-Math_b6bf8f41\/src\/main\/java\/org\/apache\/commons\/math\/stat\/clustering\/EuclideanIntegerPointjava
+++ \/tmp\/jMutRepair_Bug_dot_jar_Commons-Math_b6bf8f41\/src\/main\/java\/org\/apache\/commons\/math\/stat\/clustering\/EuclideanIntegerPointjava
@@ -82,7 +82,7 @@
 			return false;
 		}
 		for (int i = 0; i < (point.length); i++) {
-			if ((point[i]) != (otherPoint[i])) {
+			if ((this.point[i]) > (otherPoint[i])) {
 				return false;
 			}
 		}

