diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/FixMiner/Chart/3/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/FixMiner/Chart/3/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/FixMiner/Chart/3/buggy1.java	2021-08-27 19:18:33.802863950 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/FixMiner/Chart/3/tool-patch1.java	2021-08-27 19:18:33.802863950 -0700
@@ -882,7 +882,7 @@
                 this.data.remove(0);
                 removed = true;
             }
-            if (removed) {
+            if ((removed) || !(notify)) {
                 findBoundsByIteration();
                 if (notify) {
                     fireSeriesChanged();
