
--- org\/traccar\/protocol\/MeitrackProtocolDecoder.java
+++ org\/traccar\/protocol\/MeitrackProtocolDecoder.java
@@ -259,13 +259,13 @@
 			boolean started = (data[0].charAt(0)) == '0';
 			position.set("taximeterOn", started);
 			position.set("taximeterStart", data[1]);
-			if (!started) {
-				position.set("taximeterEnd", data[2]);
-				position.set("taximeterDistance", java.lang.Integer.parseInt(data[3]));
-				position.set("taximeterFare", java.lang.Integer.parseInt(data[4]));
-				position.set("taximeterTrip", data[5]);
-				position.set("taximeterWait", data[6]);
-			}
+
+
+
+
+
+
+
 		}
 
 	}

