diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/kPAR/Chart/15/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/kPAR/Chart/15/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/kPAR/Chart/15/buggy1.java	2021-08-27 19:18:35.946864010 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/kPAR/Chart/15/tool-patch1.java	2021-08-27 19:18:35.946864010 -0700
@@ -277,7 +277,10 @@
             linkH = 2 * min;
         }
         
-        PiePlotState state = initialise(g2, plotArea, this, null, info);
+        if (info == null) {
+    return;
+}
+PiePlotState state = initialise(g2, plotArea, this, null, info);
 
         // the link area defines the dog leg points for the linking lines to 
         // the labels
