diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Arja/Closure/124/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Arja/Closure/124/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Arja/Closure/124/buggy1.java	2021-08-27 19:18:32.790863921 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Arja/Closure/124/tool-patch1.java	2021-08-27 19:18:32.790863921 -0700
@@ -174,7 +174,6 @@
             // Dive down the right side of the assign.
             parent = next;
             next = leftSide.getNext();
-            break;
           } else {
             return false;
           }
