diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Closure/114/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Closure/114/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Closure/114/buggy1.java	2021-08-27 19:18:39.310864105 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Closure/114/tool-patch1.java	2021-08-27 19:18:39.310864105 -0700
@@ -117,7 +117,6 @@
       parent.replaceChild(node, block);
     } else {
       for (Node newChild : replacements) {
-        newChild.copyInformationFrom(node);
         parent.addChildBefore(newChild, node);
       }
       parent.removeChild(node);
diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Closure/114/buggy2.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Closure/114/tool-patch2.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Closure/114/buggy2.java	2021-08-27 19:18:39.306864105 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/RSRepair/Closure/114/tool-patch2.java	2021-08-27 19:18:39.310864105 -0700
@@ -1608,7 +1608,6 @@
       }
 
       if (parent.isAssign()) {
-        return scopes.get(parent);
       }
     }
 
