diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/DynaMoth/Chart/13/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/DynaMoth/Chart/13/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/DynaMoth/Chart/13/buggy1.java	2021-08-27 19:18:34.422863967 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/DynaMoth/Chart/13/tool-patch1.java	2021-08-27 19:18:34.418863967 -0700
@@ -441,6 +441,7 @@
             h[1] = size.height;
         }
         h[2] = constraint.getHeight() - h[1] - h[0];
+        if (false) {
         if (this.leftBlock != null) {
             RectangleConstraint c3 = new RectangleConstraint(0.0,
                     new Range(0.0, constraint.getWidth()),
@@ -449,6 +450,7 @@
             Size2D size = this.leftBlock.arrange(g2, c3);
             w[2] = size.width;
         }
+        }
         h[3] = h[2];
         if (this.rightBlock != null) {
             RectangleConstraint c4 = new RectangleConstraint(0.0,
