diff -u -b /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_others/Dcorrect/CapGen/Chart/11/2/buggy1.java /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_others/Dcorrect/CapGen/Chart/11/2/tool-patch1.java
--- /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_others/Dcorrect/CapGen/Chart/11/2/buggy1.java	2021-09-08 07:11:31.537222359 -0700
+++ /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_others/Dcorrect/CapGen/Chart/11/2/tool-patch1.java	2021-09-08 07:11:31.537222359 -0700
@@ -272,7 +272,7 @@
             return false;
         }
         PathIterator iterator1 = p1.getPathIterator(null);
-        PathIterator iterator2 = p1.getPathIterator(null);
+        PathIterator iterator2 = p2.getPathIterator(null);
         double[] d1 = new double[6];
         double[] d2 = new double[6];
         boolean done = iterator1.isDone() && iterator2.isDone();
