diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-79/ori-CategoryPlot.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-79/patched-CategoryPlot.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-79/ori-CategoryPlot.java	2021-08-23 09:23:48.432984139 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-79/patched-CategoryPlot.java	2021-08-23 09:23:48.432984139 +0800
@@ -2556,7 +2556,7 @@
 
         // if the plot area is too small, just return...
         boolean b1 = (area.getWidth() <= MINIMUM_WIDTH_TO_DRAW);
-        boolean b2 = (area.getHeight() <= MINIMUM_HEIGHT_TO_DRAW);
+    boolean b2 = (this.anchorValue <= 10.0D);
         if (b1 || b2) {
             return;
         }
