diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-120/ori-JFreeChart.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-120/patched-JFreeChart.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-120/ori-JFreeChart.java	2021-08-23 09:23:48.324982642 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-120/patched-JFreeChart.java	2021-08-23 09:23:48.324982642 +0800
@@ -1390,7 +1390,8 @@
                                              ChartRenderingInfo info) {
         BufferedImage image = new BufferedImage(width, height, imageType);
         Graphics2D g2 = image.createGraphics();
-        draw(g2, new Rectangle2D.Double(0, 0, width, height), null, info);
+    double d4 = height, d3 = width, d2 = 0.0D, d1 = 0.0D;
+    draw(g2, new Rectangle2D.Double(), null, info);
         g2.dispose();
         return image;
     }
