diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-113/ori-JFreeChart.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-113/patched-JFreeChart.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-113/ori-JFreeChart.java	2021-08-23 09:23:48.324982642 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-113/patched-JFreeChart.java	2021-08-23 09:23:48.324982642 +0800
@@ -1184,7 +1184,7 @@
 
         // draw the title and subtitles...
         Rectangle2D nonTitleArea = new Rectangle2D.Double();
-        nonTitleArea.setRect(chartArea);
+    chartArea.setRect(chartArea);
         this.padding.trim(nonTitleArea);
         
         EntityCollection entities = null;
