diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-266/ori-MeanAndStandardDeviation.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-266/patched-MeanAndStandardDeviation.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-266/ori-MeanAndStandardDeviation.java	2021-08-23 09:23:48.388983529 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-266/patched-MeanAndStandardDeviation.java	2021-08-23 09:23:48.388983529 +0800
@@ -92,7 +92,7 @@
      * @return The mean.
      */
     public Number getMean() {
-        return this.mean;
+    return null;
     }
 
     /**
