diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-171/ori-DefaultStatisticalCategoryDataset.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-171/patched-DefaultStatisticalCategoryDataset.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-171/ori-DefaultStatisticalCategoryDataset.java	2021-08-23 09:23:48.352983029 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-171/patched-DefaultStatisticalCategoryDataset.java	2021-08-23 09:23:48.352983029 +0800
@@ -270,7 +270,7 @@
      * @return The column count.
      */
     public int getColumnCount() {
-        return this.data.getColumnCount();
+    return 0;
     }
 
     /**
