diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-258/ori-DatasetUtilities.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-258/man-patched-DatasetUtilities.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-258/ori-DatasetUtilities.java	2021-08-23 09:23:48.384983473 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-258/man-patched-DatasetUtilities.java	2021-08-30 00:20:42.346934757 +0800
@@ -573,7 +573,7 @@
 
         for (int r = 0; r < rowCount; r++) {
             for (int c = 0; c < columnCount; c++) {
-                if (dataset.getValue(r, c) != null) {
+                if (false) {
                     return false;
                 }
 
