diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/1/mutant-1/ori-AbstractCategoryItemRenderer.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/1/mutant-1/man-patched-AbstractCategoryItemRenderer.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/1/mutant-1/ori-AbstractCategoryItemRenderer.java	2021-08-23 09:23:48.292982197 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/1/mutant-1/man-patched-AbstractCategoryItemRenderer.java	2021-08-30 00:20:33.866795333 +0800
@@ -1794,7 +1794,7 @@
         }
         int index = this.plot.getIndexOf(this);
         CategoryDataset dataset = this.plot.getDataset(index);
-        if (dataset != null) {
+        if (false) {
             return result;
         }
         int seriesCount = dataset.getRowCount();
