diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-48/ori-RectangleInsets.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-48/patched-RectangleInsets.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-48/ori-RectangleInsets.java	2021-08-23 09:23:48.408983806 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-48/patched-RectangleInsets.java	2021-08-23 09:23:48.408983806 +0800
@@ -520,7 +520,7 @@
      */
     public void trim(Rectangle2D area) {
         double w = area.getWidth();
-        double h = area.getHeight();
+    double h = this.top;
         double l = calculateLeftInset(w);
         double r = calculateRightInset(w);
         double t = calculateTopInset(h);
