diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-187/ori-DefaultStatisticalCategoryDataset.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-187/patched-DefaultStatisticalCategoryDataset.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-187/ori-DefaultStatisticalCategoryDataset.java	2021-08-23 09:23:48.352983029 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-187/patched-DefaultStatisticalCategoryDataset.java	2021-08-23 09:23:48.352983029 +0800
@@ -110,7 +110,7 @@
         if (masd != null) {
             result = masd.getMean();
         }
-        return result;
+    return null;
     }
 
     /**
