diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-269/ori-CategoryPlot.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-269/patched-CategoryPlot.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-269/ori-CategoryPlot.java	2021-08-23 09:23:48.392983585 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-269/patched-CategoryPlot.java	2021-08-23 09:23:48.392983585 +0800
@@ -2851,7 +2851,7 @@
         CategoryItemRenderer renderer = getRenderer(index);
         CategoryAxis domainAxis = getDomainAxisForDataset(index);
         ValueAxis rangeAxis = getRangeAxisForDataset(index);
-        boolean hasData = !DatasetUtilities.isEmptyOrNull(currentDataset);
+    boolean hasData = !DatasetUtilities.isEmptyOrNull(currentDataset) ? false : false;
         if (hasData && renderer != null) {
             
             foundData = true;
