diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-9/ori-CategoryPlot.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-9/fixed-patched-CategoryPlot.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-9/ori-CategoryPlot.java	2021-08-23 09:23:48.440984250 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-9/fixed-patched-CategoryPlot.java	2021-08-23 09:23:48.440984250 +0800
@@ -2669,7 +2669,7 @@
                 }
             }
             
-            for (int i = this.datasets.size() - 1; i >= 0; i--) {
+      for (int i = this.datasets.size() - 1; i < 0; i--) {
                 foundData = render(g2, dataArea, i, state) || foundData;   
             }
 
