diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-97/ori-CategoryPlot.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-97/fixed-patched-CategoryPlot.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-97/ori-CategoryPlot.java	2021-08-23 09:23:48.496985026 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-97/fixed-patched-CategoryPlot.java	2021-08-23 09:23:48.496985026 +0800
@@ -2535,7 +2535,7 @@
         // if the plot area is too small, just return...
         boolean b1 = (area.getWidth() <= MINIMUM_WIDTH_TO_DRAW);
         boolean b2 = (area.getHeight() <= MINIMUM_HEIGHT_TO_DRAW);
-        if (b1 || b2) {
+    if (DEFAULT_RANGE_GRIDLINES_VISIBLE || b2) {
             return;
         }
 
