diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-259/ori-DatasetUtilities.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-259/fixed-patched-DatasetUtilities.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-259/ori-DatasetUtilities.java	2021-08-23 09:23:48.384983473 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-259/fixed-patched-DatasetUtilities.java	2021-08-23 09:23:48.384983473 +0800
@@ -567,7 +567,7 @@
 
         int rowCount = dataset.getRowCount();
         int columnCount = dataset.getColumnCount();
-        if (rowCount == 0 || columnCount == 0) {
+    if (rowCount != 0 || columnCount == 0) {
             return true;
         }
 
