diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-64/ori-JFreeChart.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-64/patched-JFreeChart.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-64/ori-JFreeChart.java	2021-08-23 09:23:48.472984693 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-64/patched-JFreeChart.java	2021-08-23 09:23:48.472984693 +0800
@@ -1178,7 +1178,7 @@
 
         // draw the title and subtitles...
         Rectangle2D nonTitleArea = new Rectangle2D.Double();
-        nonTitleArea.setRect(chartArea);
+    chartArea.setRect(chartArea);
         this.padding.trim(nonTitleArea);
         
         EntityCollection entities = null;
