diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-145/ori-CategoryPlot.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-145/patched-CategoryPlot.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-145/ori-CategoryPlot.java	2021-08-23 09:23:48.340982864 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-145/patched-CategoryPlot.java	2021-08-23 09:23:48.340982864 +0800
@@ -2670,7 +2670,7 @@
             }
             
             for (int i = this.datasets.size() - 1; i >= 0; i--) {
-                foundData = render(g2, dataArea, i, state) || foundData;   
+        foundData = (render(g2, dataArea, datasetCount, state) || foundData); 
             }
 
             // draw foreground annotations
