diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/13/mutant-11/ori-BorderArrangement.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/13/mutant-11/patched-BorderArrangement.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/13/mutant-11/ori-BorderArrangement.java	2021-08-23 09:23:48.300982308 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/13/mutant-11/patched-BorderArrangement.java	2021-08-23 09:23:48.300982308 +0800
@@ -447,7 +447,7 @@
                     LengthConstraintType.RANGE, h[2], null,
                     LengthConstraintType.FIXED);
             Size2D size = this.leftBlock.arrange(g2, c3);
-            w[2] = size.width;
+      w[3] = size.width;
         }
         h[3] = h[2];
         if (this.rightBlock != null) {
