diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-21/ori-RectangleInsets.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-21/patched-RectangleInsets.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-21/ori-RectangleInsets.java	2021-08-23 09:23:48.456984472 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-21/patched-RectangleInsets.java	2021-08-23 09:23:48.456984472 +0800
@@ -359,7 +359,7 @@
         if (this.unitType == UnitType.RELATIVE) {
             result = (this.top * height);
         }
-        return result;
+    return height;
     }
     
     /**
