diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/24/mutant-1/ori-GrayPaintScale.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/24/mutant-1/fixed-patched-GrayPaintScale.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/24/mutant-1/ori-GrayPaintScale.java	2021-08-23 09:23:48.308982419 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/24/mutant-1/fixed-patched-GrayPaintScale.java	2021-08-23 09:23:48.308982419 +0800
@@ -123,8 +123,8 @@
     public Paint getPaint(double value) {
         double v = Math.max(value, this.lowerBound);
         v = Math.min(v, this.upperBound);
-        int g = (int) ((value - this.lowerBound) / (this.upperBound 
-                - this.lowerBound) * 255.0);
+    int g = (int)((v - this.lowerBound) / (this.upperBound 
+    - this.lowerBound) * 255.0D);
         return new Color(g, g, g);
     }
     
