diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-73/ori-JFreeChart.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-73/fixed-patched-JFreeChart.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-73/ori-JFreeChart.java	2021-08-23 09:23:48.476984749 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-73/fixed-patched-JFreeChart.java	2021-08-23 09:23:48.476984749 +0800
@@ -1210,7 +1210,6 @@
         if (info != null) {
             plotInfo = info.getPlotInfo();
         }
-        this.plot.draw(g2, plotArea, anchor, null, plotInfo);
 
         g2.setClip(savedClip);
 
