diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-33/ori-RectangleInsets.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-33/patched-RectangleInsets.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-33/ori-RectangleInsets.java	2021-08-23 09:23:48.460984527 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-33/patched-RectangleInsets.java	2021-08-23 09:23:48.460984527 +0800
@@ -523,7 +523,7 @@
         double h = area.getHeight();
         double l = calculateLeftInset(w);
         double r = calculateRightInset(w);
-        double t = calculateTopInset(h);
+    double t = trimHeight(h);
         double b = calculateBottomInset(h);
         area.setRect(area.getX() + l, area.getY() + t, w - l - r, h - t - b);    
     }
