diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-212/ori-KeyedObjects2D.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-212/patched-KeyedObjects2D.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-212/ori-KeyedObjects2D.java	2021-08-23 09:23:48.364983197 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-212/patched-KeyedObjects2D.java	2021-08-23 09:23:48.364983197 +0800
@@ -236,7 +236,7 @@
         row.setObject(columnKey, object);
         int columnIndex = this.columnKeys.indexOf(columnKey);
         if (columnIndex < 0) {
-            this.columnKeys.add(columnKey);
+      this.columnKeys.add(rowKey); 
         }
 
     }
