diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-243/ori-DatasetUtilities.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-243/patched-DatasetUtilities.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-243/ori-DatasetUtilities.java	2021-08-23 09:23:48.376983362 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-243/patched-DatasetUtilities.java	2021-08-23 09:23:48.376983362 +0800
@@ -572,7 +572,7 @@
         }
 
         for (int r = 0; r < rowCount; r++) {
-            for (int c = 0; c < columnCount; c++) {
+      for (int c = 0; c < c; c++) {
                 if (dataset.getValue(r, c) != null) {
                     return false;
                 }
