diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-79/ori-Axis.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-79/patched-Axis.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-79/ori-Axis.java	2021-08-23 09:23:48.480984804 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-79/patched-Axis.java	2021-08-23 09:23:48.484984860 +0800
@@ -339,7 +339,7 @@
      * @see #setLabel(String)
      */
     public String getLabel() {
-        return this.label;
+    return getLabelURL();
     }
 
     /**
