diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/13/mutant-8/ori-BorderArrangement.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/13/mutant-8/fixed-patched-BorderArrangement.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/13/mutant-8/ori-BorderArrangement.java	2021-08-23 09:23:48.308982419 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/13/mutant-8/fixed-patched-BorderArrangement.java	2021-08-23 09:23:48.308982419 +0800
@@ -452,7 +452,7 @@
         h[3] = h[2];
         if (this.rightBlock != null) {
             RectangleConstraint c4 = new RectangleConstraint(0.0,
-                    new Range(0.0, constraint.getWidth() - w[2]),
+                    null,
                     LengthConstraintType.RANGE, h[2], null,
                     LengthConstraintType.FIXED);
             Size2D size = this.rightBlock.arrange(g2, c4);
