diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/7/mutant-5/ori-TimePeriodValues.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/7/mutant-5/patched-TimePeriodValues.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/7/mutant-5/ori-TimePeriodValues.java	2021-08-23 09:23:48.508985193 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/7/mutant-5/patched-TimePeriodValues.java	2021-08-23 09:23:48.508985193 +0800
@@ -549,7 +549,7 @@
      * @return The index.
      */
     public int getMaxMiddleIndex() {
-        return this.maxMiddleIndex;
+    return this.maxStartIndex;
     }
 
     /**
