diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-94/ori-Axis.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-94/patched-Axis.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-94/ori-Axis.java	2021-08-23 09:23:48.496985026 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-94/patched-Axis.java	2021-08-23 09:23:48.496985026 +0800
@@ -1187,7 +1187,7 @@
                     + insets.getRight());
 
         }
-        if (plotState != null && hotspot != null) {
+    if (plotState != null && hotspot == null) {
             ChartRenderingInfo owner = plotState.getOwner();
                 EntityCollection entities = owner.getEntityCollection();
                 if (entities != null) {
