diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-95/ori-Axis.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-95/patched-Axis.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-95/ori-Axis.java	2021-08-23 09:23:48.496985026 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-95/patched-Axis.java	2021-08-23 09:23:48.496985026 +0800
@@ -311,7 +311,7 @@
      * @see #setVisible(boolean)
      */
     public boolean isVisible() {
-        return this.visible;
+    return false;
     }
 
     /**
