diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-65/ori-Axis.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-65/patched-Axis.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-65/ori-Axis.java	2021-08-23 09:23:48.472984693 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-65/patched-Axis.java	2021-08-23 09:23:48.472984693 +0800
@@ -339,7 +339,7 @@
      * @see #setLabel(String)
      */
     public String getLabel() {
-        return this.label;
+    return this.labelURL;
     }
 
     /**
