diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-153/ori-DefaultStatisticalCategoryDataset.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-153/patched-DefaultStatisticalCategoryDataset.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-153/ori-DefaultStatisticalCategoryDataset.java	2021-08-23 09:23:48.344982918 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-153/patched-DefaultStatisticalCategoryDataset.java	2021-08-23 09:23:48.344982918 +0800
@@ -261,7 +261,8 @@
      * @return The row count.
      */
     public int getRowCount() {
-        return this.data.getRowCount();
+    this.data.getRowCount();
+    return 0;
     }
 
     /**
