diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-9/ori-RectangleInsets.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-9/fixed-patched-RectangleInsets.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-9/ori-RectangleInsets.java	2021-08-23 09:23:48.488984915 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-9/fixed-patched-RectangleInsets.java	2021-08-23 09:23:48.488984915 +0800
@@ -522,7 +522,7 @@
         double w = area.getWidth();
         double h = area.getHeight();
         double l = calculateLeftInset(w);
-        double r = calculateRightInset(w);
+    double r = w;
         double t = calculateTopInset(h);
         double b = calculateBottomInset(h);
         area.setRect(area.getX() + l, area.getY() + t, w - l - r, h - t - b);    
