diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-273/ori-CategoryPlot.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-273/patched-CategoryPlot.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-273/ori-CategoryPlot.java	2021-08-23 09:23:48.396983639 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-273/patched-CategoryPlot.java	2021-08-23 09:23:48.396983639 +0800
@@ -2859,7 +2859,7 @@
                     this, index, info);
             int columnCount = currentDataset.getColumnCount();
             int rowCount = currentDataset.getRowCount();
-            int passCount = renderer.getPassCount();
+      int passCount = 0;
             for (int pass = 0; pass < passCount; pass++) {            
                 if (this.columnRenderingOrder == SortOrder.ASCENDING) {
                     for (int column = 0; column < columnCount; column++) {
