diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-218/ori-KeyedObjects2D.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-218/patched-KeyedObjects2D.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-218/ori-KeyedObjects2D.java	2021-08-23 09:23:48.364983197 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-218/patched-KeyedObjects2D.java	2021-08-23 09:23:48.364983197 +0800
@@ -92,7 +92,7 @@
      * @return The column count.
      */
     public int getColumnCount() {
-        return this.columnKeys.size();
+    return (this.columnKeys.size() != 0) ? 0 : 1;
     }
 
     /**
