diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-87/ori-JFreeChart.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-87/fixed-patched-JFreeChart.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-87/ori-JFreeChart.java	2021-08-23 09:23:48.436984194 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/25/mutant-87/fixed-patched-JFreeChart.java	2021-08-23 09:23:48.436984194 +0800
@@ -1370,8 +1370,7 @@
      */
     public BufferedImage createBufferedImage(int width, int height, 
                                              ChartRenderingInfo info) {
-        return createBufferedImage(width, height, BufferedImage.TYPE_INT_RGB, 
-                info);
+    return null;
     }
 
     /**
