diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-39/ori-RectangleInsets.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-39/patched-RectangleInsets.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-39/ori-RectangleInsets.java	2021-08-23 09:23:48.460984527 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Chart/26/mutant-39/patched-RectangleInsets.java	2021-08-23 09:23:48.460984527 +0800
@@ -389,7 +389,7 @@
         if (this.unitType == UnitType.RELATIVE) {
             result = (this.bottom * height);
         }
-        return result;
+    return height;
     }
 
     /**
