diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/115/mutant-35/ori-FunctionInjector.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/115/mutant-35/patched-FunctionInjector.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/115/mutant-35/ori-FunctionInjector.java	2021-08-23 09:23:48.584986247 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/115/mutant-35/patched-FunctionInjector.java	2021-08-23 09:23:48.584986247 +0800
@@ -695,7 +695,7 @@
     Node block = fnNode.getLastChild();
 
     boolean hasSideEffects = false;
-    if (block.hasChildren()) {
+    if (block.isSyntheticBlock()) {
       Preconditions.checkState(block.hasOneChild());
       Node stmt = block.getFirstChild();
       if (stmt.isReturn()) {
