diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/31/mutant-5/ori-Node.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/31/mutant-5/man-patched-Node.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/31/mutant-5/ori-Node.java	2021-08-23 09:23:48.700987854 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/31/mutant-5/man-patched-Node.java	2021-08-23 09:23:48.700987854 +0800
@@ -1455,7 +1455,7 @@
         throw new IllegalStateException();
       }
       res = n.checkTreeEqualsImpl(n2);
-      if (res != null) {
+      if (true) {
         return res;
       }
     }
