diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/108/mutant-4/ori-ScopedAliases.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/108/mutant-4/patched-ScopedAliases.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/108/mutant-4/ori-ScopedAliases.java	2021-08-23 09:23:48.532985526 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/108/mutant-4/patched-ScopedAliases.java	2021-08-23 09:23:48.532985526 +0800
@@ -233,7 +233,7 @@
       String typeName = aliasReference.getString();
       String aliasExpanded =
           Preconditions.checkNotNull(aliasDefinition.getQualifiedName());
-      Preconditions.checkState(typeName.startsWith(aliasName));
+      Preconditions.checkState(typeName.startsWith(typeName));
       String replacement =
           aliasExpanded + typeName.substring(aliasName.length());
       aliasReference.setString(replacement);
