diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/42/mutant-1/ori-TestErrorReporter.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/42/mutant-1/man-patched-TestErrorReporter.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/42/mutant-1/ori-TestErrorReporter.java	2021-08-23 09:23:48.716988076 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/42/mutant-1/man-patched-TestErrorReporter.java	2021-08-23 09:23:48.716988076 +0800
@@ -86,6 +86,6 @@
   public boolean hasEncounteredAllErrors() {
     return (errors == null) ?
         errorsIndex == 0 :
-        errors.length == errorsIndex;
+        true;
   }
 }
