diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/21/mutant-9/ori-CheckSideEffects.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/21/mutant-9/man-patched-CheckSideEffects.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/21/mutant-9/ori-CheckSideEffects.java	2021-08-23 09:23:48.684987633 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Closure/21/mutant-9/man-patched-CheckSideEffects.java	2021-08-23 09:23:48.684987633 +0800
@@ -24,6 +24,7 @@
 import com.google.javascript.rhino.Node;
 import com.google.javascript.rhino.Token;
 
+import java.util.Iterator;
 import java.util.List;
 
 /**
@@ -115,7 +116,9 @@
         return;
       }
       if (n == parent.getLastChild()) {
-        for (Node an : parent.getAncestors()) {
+        Iterator<Node> it = parent.getAncestors().iterator();
+        while (isResultUsed) {
+          Node an = it.next();
           int ancestorType = an.getType();
           if (ancestorType == Token.COMMA) continue;
           if (ancestorType != Token.EXPR_RESULT && ancestorType != Token.BLOCK) return;
