diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/63/mutant-10/ori-DurationFormatUtils.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/63/mutant-10/man-patched-DurationFormatUtils.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/63/mutant-10/ori-DurationFormatUtils.java	2021-08-23 09:23:48.852989961 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/63/mutant-10/man-patched-DurationFormatUtils.java	2021-08-23 09:23:48.852989961 +0800
@@ -435,7 +435,6 @@
         int startValue = start.get(field);
         if (endValue < startValue) {
             int newdiff = startValue - endValue;
-            end.add( field, newdiff );
             return newdiff;
         } else {
             return 0;
