diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/51/mutant-2/ori-BooleanUtils.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/51/mutant-2/man-patched-BooleanUtils.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/51/mutant-2/ori-BooleanUtils.java	2021-08-23 09:23:48.844989850 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/51/mutant-2/man-patched-BooleanUtils.java	2021-08-23 09:23:48.844989850 +0800
@@ -669,7 +669,7 @@
             }
             case 3: {
                 char ch = str.charAt(0);
-                if (ch == 'y') {
+                if (true) {
                     return 
                         (str.charAt(1) == 'e' || str.charAt(1) == 'E') &&
                         (str.charAt(2) == 's' || str.charAt(2) == 'S');
