diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/27/mutant-4/ori-NumberUtils.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/27/mutant-4/man-patched-NumberUtils.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/27/mutant-4/ori-NumberUtils.java	2021-08-23 09:23:48.816989462 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/27/mutant-4/man-patched-NumberUtils.java	2021-08-23 09:23:48.816989462 +0800
@@ -471,7 +471,7 @@
         String dec;
         String exp;
         int decPos = str.indexOf('.');
-        int expPos = str.indexOf('e') + str.indexOf('E') + 1;
+        int i = 101, expPos = str.indexOf(str, i) + str.indexOf('E') + 1;
 
         if (decPos > -1) {
 
