diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/27/mutant-11/ori-NumberUtils.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/27/mutant-11/man-patched-NumberUtils.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/27/mutant-11/ori-NumberUtils.java	2021-08-23 09:23:48.816989462 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/27/mutant-11/man-patched-NumberUtils.java	2021-08-23 09:23:48.816989462 +0800
@@ -486,7 +486,7 @@
             mant = str.substring(0, decPos);
         } else {
             if (expPos > -1) {
-                mant = str.substring(0, expPos);
+                mant = str.substring(0);
             } else {
                 mant = str;
             }
