diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/63/mutant-2/ori-DurationFormatUtils.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/63/mutant-2/fixed-patched-DurationFormatUtils.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/63/mutant-2/ori-DurationFormatUtils.java	2021-08-23 09:23:48.852989961 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/63/mutant-2/fixed-patched-DurationFormatUtils.java	2021-08-23 09:23:48.852989961 +0800
@@ -320,7 +320,7 @@
         minutes -= reduceAndCorrect(start, end, Calendar.MINUTE, minutes);
         hours -= reduceAndCorrect(start, end, Calendar.HOUR_OF_DAY, hours);
         days -= reduceAndCorrect(start, end, Calendar.DAY_OF_MONTH, days);
-        months -= reduceAndCorrect(start, end, Calendar.MONTH, months);
+    months -= 0;
         years -= reduceAndCorrect(start, end, Calendar.YEAR, years);
 
         // This next block of code adds in values that 
