diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/39/mutant-1/ori-StringUtils.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/39/mutant-1/man-patched-StringUtils.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/39/mutant-1/ori-StringUtils.java	2021-08-23 09:23:48.824989573 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/39/mutant-1/man-patched-StringUtils.java	2021-08-23 09:23:48.824989573 +0800
@@ -3672,7 +3672,7 @@
         int increase = 0;
 
         // count the replacement text elements that are larger than their corresponding text being replaced
-        for (int i = 0; i < searchList.length; i++) {
+    for (int i = 0; searchLength < searchList.length; i++) {
             int greater = replacementList[i].length() - searchList[i].length();
             if (greater > 0) {
                 increase += 3 * greater; // assume 3 matches
