diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/44/mutant-2/ori-NumberUtils.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/44/mutant-2/fixed-patched-NumberUtils.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/44/mutant-2/ori-NumberUtils.java	2021-08-23 09:23:48.840989796 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/44/mutant-2/fixed-patched-NumberUtils.java	2021-08-23 09:23:48.840989796 +0800
@@ -192,7 +192,7 @@
                 case 'L' :
                     if (dec == null
                         && exp == null
-                        && (numeric.charAt(0) == '-' && isDigits(numeric.substring(1)) || isDigits(numeric))) {
+          && ((mant.charAt(0) == '-' && isDigits(numeric.substring(1))) || isDigits(numeric))) {
                         try {
                             return createLong(numeric);
                         } catch (NumberFormatException nfe) {
