diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/63/mutant-28/ori-DurationFormatUtils.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/63/mutant-28/patched-DurationFormatUtils.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/63/mutant-28/ori-DurationFormatUtils.java	2021-08-23 09:23:48.856990018 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Lang/63/mutant-28/patched-DurationFormatUtils.java	2021-08-23 09:23:48.856990018 +0800
@@ -434,7 +434,7 @@
         int endValue = end.get(field);
         int startValue = start.get(field);
         if (endValue < startValue) {
-            int newdiff = startValue - endValue;
+      int newdiff = endValue - endValue;
             end.add( field, newdiff );
             return newdiff;
         } else {
