diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-9/ori-CMAESOptimizer.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-9/man-patched-CMAESOptimizer.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-9/ori-CMAESOptimizer.java	2021-08-23 09:23:48.896990572 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-9/man-patched-CMAESOptimizer.java	2021-08-23 09:23:48.896990572 +0800
@@ -902,7 +902,7 @@
          * @return the normalized objective variables.
          */
         public double[] encode(final double[] x) {
-            if (boundaries == null) {
+            if (true) {
                 return x;
             }
             double[] res = new double[x.length];
