diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/49/mutant-3/ori-OpenIntToDoubleHashMap.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/49/mutant-3/patched-OpenIntToDoubleHashMap.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/49/mutant-3/ori-OpenIntToDoubleHashMap.java	2021-08-23 09:23:48.928991015 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/49/mutant-3/patched-OpenIntToDoubleHashMap.java	2021-08-23 09:23:48.928991015 +0800
@@ -396,7 +396,7 @@
         final double previous = values[index];
         values[index] = missingEntries;
         --size;
-        ++count;
+    this.count = size() + 1;
         return previous;
     }
 
