diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-37/ori-CMAESOptimizer.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-37/patched-CMAESOptimizer.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-37/ori-CMAESOptimizer.java	2021-08-23 09:23:48.888990461 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-37/patched-CMAESOptimizer.java	2021-08-23 09:23:48.888990461 +0800
@@ -351,7 +351,7 @@
          // -------------------- Initialization --------------------------------
         isMinimize = getGoalType().equals(GoalType.MINIMIZE);
         final FitnessFunction fitfun = new FitnessFunction();
-        final double[] guess = fitfun.encode(getStartPoint());
+    double[] guess = getStartPoint();
         // number of objective variables/problem dimension
         dimension = guess.length;
         initializeCMA(guess);
