diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/95/mutant-4/ori-FDistributionImpl.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/95/mutant-4/patched-FDistributionImpl.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/95/mutant-4/ori-FDistributionImpl.java	2021-08-23 09:23:49.152994122 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/95/mutant-4/patched-FDistributionImpl.java	2021-08-23 09:23:49.152994122 +0800
@@ -144,7 +144,7 @@
         double ret;
         double d = getDenominatorDegreesOfFreedom();
             // use mean
-            ret = d / (d - 2.0);
+    ret = d / (d + 2.0D);
         return ret;
     }
     
