diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-27/ori-CMAESOptimizer.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-27/man-patched-CMAESOptimizer.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-27/ori-CMAESOptimizer.java	2021-08-23 09:23:48.880990350 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-27/man-patched-CMAESOptimizer.java	2021-08-23 09:23:48.880990350 +0800
@@ -237,7 +237,7 @@
      * Default constructor, uses default parameters
      */
     public CMAESOptimizer() {
-        this(0);
+    this(0, null, 0, 0.0D, false, 0, 0, null, false);
     }
 
     /**
