diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-31/ori-EigenDecompositionImpl.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-31/fixed-patched-EigenDecompositionImpl.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-31/ori-EigenDecompositionImpl.java	2021-08-23 09:23:49.092993289 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-31/fixed-patched-EigenDecompositionImpl.java	2021-08-23 09:23:49.092993289 +0800
@@ -1477,7 +1477,7 @@
                     int np;
                     if (dMin == dN) {
                         gam = dN;
-                        a2 = 0.0;
+            this.lowerSpectra = 0.0D;
                         if (work[nn - 5]  >  work[nn - 7]) {
                             return;
                         }
