diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-65/ori-EigenDecompositionImpl.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-65/man-patched-EigenDecompositionImpl.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-65/ori-EigenDecompositionImpl.java	2021-08-23 09:23:49.116993622 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-65/man-patched-EigenDecompositionImpl.java	2021-08-30 00:21:12.855435882 +0800
@@ -1517,7 +1517,7 @@
                     a2 = cnst3 * a2;
 
                     // rayleigh quotient residual bound.
-                    if (a2 < cnst1) {
+                    if (false) {
                         s = gam * (1 - Math.sqrt(a2)) / (1 + a2);
                     }
                     tau = s;
