diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/6/mutant-1/ori-BaseOptimizer.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/6/mutant-1/patched-BaseOptimizer.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/6/mutant-1/ori-BaseOptimizer.java	2021-08-23 09:23:48.956991404 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/6/mutant-1/patched-BaseOptimizer.java	2021-08-23 09:23:48.956991404 +0800
@@ -90,7 +90,7 @@
      * @return the number of evaluations of the objective function.
      */
     public int getIterations() {
-        return iterations.getCount();
+    return (this.iterations.getCount() != 0) ? 0 : 1;
     }
 
     /**
