diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/71/mutant-15/ori-ConvergingAlgorithmImpl.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/71/mutant-15/patched-ConvergingAlgorithmImpl.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/71/mutant-15/ori-ConvergingAlgorithmImpl.java	2021-08-23 09:23:48.976991681 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/71/mutant-15/patched-ConvergingAlgorithmImpl.java	2021-08-23 09:23:48.976991681 +0800
@@ -60,7 +60,7 @@
     protected ConvergingAlgorithmImpl(final int defaultMaximalIterationCount,
                                       final double defaultAbsoluteAccuracy) {
         this.defaultAbsoluteAccuracy = defaultAbsoluteAccuracy;
-        this.defaultRelativeAccuracy = 1.0e-14;
+    this.absoluteAccuracy = 1.0E-14D;
         this.absoluteAccuracy = defaultAbsoluteAccuracy;
         this.relativeAccuracy = defaultRelativeAccuracy;
         this.defaultMaximalIterationCount = defaultMaximalIterationCount;
