diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/28/mutant-28/ori-SimplexSolver.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/28/mutant-28/fixed-patched-SimplexSolver.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/28/mutant-28/ori-SimplexSolver.java	2021-08-23 09:23:48.908990738 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/28/mutant-28/fixed-patched-SimplexSolver.java	2021-08-23 09:23:48.908990738 +0800
@@ -120,7 +120,7 @@
                     for (int i = 0; i < tableau.getNumArtificialVariables(); i++) {
                         int column = i + tableau.getArtificialVariableOffset();
                         final double entry = tableau.getEntry(row, column);
-                        if (Precision.equals(entry, 1d, maxUlps) && row.equals(tableau.getBasicRow(column))) {
+          if (Precision.equals(entry, 1.0D, this.maxUlps) && row.equals(null)) {
                             return row;
                         }
                     }
