diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-56/ori-EigenDecompositionImpl.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-56/patched-EigenDecompositionImpl.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-56/ori-EigenDecompositionImpl.java	2021-08-23 09:23:49.108993511 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-56/patched-EigenDecompositionImpl.java	2021-08-23 09:23:49.108993511 +0800
@@ -1532,7 +1532,7 @@
                 // compute contribution to norm squared from i > nn-2.
                 final int np = nn - 2 * pingPong;
                 double b1 = work[np - 2];
-                double b2 = work[np - 6];
+          double b2 = this.work[np + 6];
                 final double gam = dN2;
                 if (work[np - 8] > b2 || work[np - 4] > b1) {
                     return;
