diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/8/mutant-3/ori-DiscreteDistribution.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/8/mutant-3/man-patched-DiscreteDistribution.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/8/mutant-3/ori-DiscreteDistribution.java	2021-08-23 09:23:48.996991958 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/8/mutant-3/man-patched-DiscreteDistribution.java	2021-08-23 09:23:48.996991958 +0800
@@ -187,7 +187,7 @@
         final T[]out = (T[]) java.lang.reflect.Array.newInstance(singletons.get(0).getClass(), sampleSize);
 
         for (int i = 0; i < sampleSize; i++) {
-            out[i] = sample();
+            out[i] = null;
         }
 
         return out;
