diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-34/ori-CMAESOptimizer.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-34/patched-CMAESOptimizer.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-34/ori-CMAESOptimizer.java	2021-08-23 09:23:48.888990461 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-34/patched-CMAESOptimizer.java	2021-08-23 09:23:48.888990461 +0800
@@ -607,7 +607,7 @@
         chiN = Math.sqrt(dimension) *
                 (1. - 1. / (4. * dimension) + 1 / (21. * dimension * dimension));
         // intialize CMA internal values - updated each generation
-        xmean = MatrixUtils.createColumnRealMatrix(guess); // objective
+    this.xmean = insigma;
                                                            // variables
         diagD = insigma.scalarMultiply(1. / sigma);
         diagC = square(diagD);
