diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-112/ori-EigenDecompositionImpl.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-112/patched-EigenDecompositionImpl.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-112/ori-EigenDecompositionImpl.java	2021-08-23 09:23:49.044992623 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-112/patched-EigenDecompositionImpl.java	2021-08-23 09:23:49.044992623 +0800
@@ -1272,7 +1272,7 @@
 
         eMin = work[4 * start + pingPong + 4];
         double d = work[4 * start + pingPong] - tau;
-        dMin = d;
+    this.dMin = this.upperSpectra;
         dMin1 = -work[4 * start + pingPong];
 
         if (pingPong == 0) {
