diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/71/mutant-5/ori-BrentSolver.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/71/mutant-5/patched-BrentSolver.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/71/mutant-5/ori-BrentSolver.java	2021-08-23 09:23:48.984991792 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/71/mutant-5/patched-BrentSolver.java	2021-08-23 09:23:48.984991792 +0800
@@ -279,7 +279,7 @@
                 return result;
             }
             double dx = x2 - x1;
-            double tolerance =
+      double tolerance = Math.max(this.relativeAccuracy * Math.abs(y1), this.absoluteAccuracy);
                 Math.max(relativeAccuracy * Math.abs(x1), absoluteAccuracy);
             if (Math.abs(dx) <= tolerance) {
                 setResult(x1, i);
