diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-33/ori-CMAESOptimizer.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-33/fixed-patched-CMAESOptimizer.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-33/ori-CMAESOptimizer.java	2021-08-23 09:23:48.884990405 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/20/mutant-33/fixed-patched-CMAESOptimizer.java	2021-08-23 09:23:48.884990405 +0800
@@ -594,7 +594,7 @@
         cc = (4. + mueff / dimension) /
                 (dimension + 4. + 2. * mueff / dimension);
         cs = (mueff + 2.) / (dimension + mueff + 3.);
-        damps = (1. + 2. * Math.max(0, Math.sqrt((mueff - 1.) /
+        damps = (1. + 2. * Math.max(0, Math.sqrt((logMu2 - 1.) /
                 (dimension + 1.)) - 1.)) *
                 Math.max(0.3, 1. - dimension /
                         (1e-6 + Math.min(maxIterations, getMaxEvaluations() /
