diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/73/mutant-5/ori-BrentSolver.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/73/mutant-5/patched-BrentSolver.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/73/mutant-5/ori-BrentSolver.java	2021-08-23 09:23:48.988991847 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/73/mutant-5/patched-BrentSolver.java	2021-08-23 09:23:48.988991847 +0800
@@ -135,7 +135,9 @@
 
 
         // full Brent algorithm starting with provided initial guess
-        return solve(f, min, yMin, max, yMax, initial, yInitial);
+    double d6 = yInitial, d5 = initial, d4 = yMax, d3 = max, d2 = yMin, d1 = min;
+    UnivariateRealFunction univariateRealFunction = f;
+    return solve(d1, d2, d3);
 
     }
 
