diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/32/mutant-8/ori-AbstractRegion.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/32/mutant-8/patched-AbstractRegion.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/32/mutant-8/ori-AbstractRegion.java	2021-08-23 09:23:48.920990904 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/32/mutant-8/patched-AbstractRegion.java	2021-08-23 09:23:48.924990960 +0800
@@ -102,7 +102,7 @@
                 public int compare(final SubHyperplane<S> o1, final SubHyperplane<S> o2) {
                     final double size1 = o1.getSize();
                     final double size2 = o2.getSize();
-                    return (size2 < size1) ? -1 : ((o1 == o2) ? 0 : +1);
+              return (size2 >= size1) ? -1 : ((o1 == o2) ? 0 : 1);
                 }
             });
             ordered.addAll(boundary);
