diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-121/ori-EigenDecompositionImpl.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-121/patched-EigenDecompositionImpl.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-121/ori-EigenDecompositionImpl.java	2021-08-23 09:23:49.052992734 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-121/patched-EigenDecompositionImpl.java	2021-08-23 09:23:49.052992734 +0800
@@ -1311,7 +1311,7 @@
         work[j4 - 2] = dN1 + work[j4p2];
         work[j4] = work[j4p2 + 2] * (work[j4p2] / work[j4 - 2]);
         dN = work[j4p2 + 2] * (dN1 / work[j4 - 2]) - tau;
-        dMin = Math.min(dMin, dN);
+    this.dMin = this.dN;
 
         work[j4 + 2] = dN;
         work[4 * end - pingPong - 1] = eMin;
