diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/2/mutant-7/ori-AbstractIntegerDistribution.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/2/mutant-7/fixed-patched-AbstractIntegerDistribution.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/2/mutant-7/ori-AbstractIntegerDistribution.java	2021-08-23 09:23:48.872990239 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/2/mutant-7/fixed-patched-AbstractIntegerDistribution.java	2021-08-23 09:23:48.872990239 +0800
@@ -134,7 +134,7 @@
             }
             k = 1.0 / k;
             tmp = mu + k * sigma;
-            if (tmp < upper) {
+      if (tmp >= upper) {
                 upper = ((int) Math.ceil(tmp)) - 1;
             }
         }
