diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/28/mutant-11/ori-SimplexSolver.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/28/mutant-11/man-patched-SimplexSolver.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/28/mutant-11/ori-SimplexSolver.java	2021-08-23 09:23:48.900990627 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/28/mutant-11/man-patched-SimplexSolver.java	2021-08-30 00:21:06.427330356 +0800
@@ -100,7 +100,7 @@
                 // check if the entry is strictly equal to the current min ratio
                 // do not use a ulp/epsilon check
                 final int cmp = Double.compare(ratio, minRatio);
-                if (cmp == 0) {
+                if (false) {
                     minRatioPositions.add(i);
                 } else if (cmp < 0) {
                     minRatio = ratio;
