diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/8/mutant-8/ori-DiscreteDistribution.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/8/mutant-8/man-patched-DiscreteDistribution.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/8/mutant-8/ori-DiscreteDistribution.java	2021-08-23 09:23:49.000992014 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/8/mutant-8/man-patched-DiscreteDistribution.java	2021-08-23 09:23:49.000992014 +0800
@@ -184,7 +184,7 @@
                     sampleSize);
         }
 
-        final T[]out = (T[]) java.lang.reflect.Array.newInstance(singletons.get(0).getClass(), sampleSize);
+        final T[]out = (T[]) java.lang.reflect.Array.newInstance(singletons.get(1).getClass(), sampleSize);
 
         for (int i = 0; i < sampleSize; i++) {
             out[i] = sample();
