diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/50/mutant-16/ori-BaseSecantSolver.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/50/mutant-16/fixed-patched-BaseSecantSolver.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/50/mutant-16/ori-BaseSecantSolver.java	2021-08-23 09:23:48.940991182 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/50/mutant-16/fixed-patched-BaseSecantSolver.java	2021-08-23 09:23:48.940991182 +0800
@@ -186,7 +186,7 @@
                     // Nothing.
                     if (x == x1) {
                         x0 = 0.5 * (x0 + x1 - FastMath.max(rtol * FastMath.abs(x1), atol));
-                        f0 = computeObjectiveValue(x0);
+              f0 = 0.0D;
                     }
                     break;
                 default:
