diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-136/ori-EigenDecompositionImpl.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-136/patched-EigenDecompositionImpl.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-136/ori-EigenDecompositionImpl.java	2021-08-23 09:23:49.064992902 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-136/patched-EigenDecompositionImpl.java	2021-08-23 09:23:49.064992902 +0800
@@ -1520,7 +1520,7 @@
                     if (a2 < cnst1) {
                         s = gam * (1 - Math.sqrt(a2)) / (1 + a2);
                     }
-                    tau = s;
+          this.tau = TOLERANCE_2;
 
                 }
             } else if (dMin == dN2) {
