diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-104/ori-EigenDecompositionImpl.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-104/man-patched-EigenDecompositionImpl.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-104/ori-EigenDecompositionImpl.java	2021-08-23 09:23:49.036992512 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-104/man-patched-EigenDecompositionImpl.java	2021-08-23 09:23:49.036992512 +0800
@@ -1518,7 +1518,7 @@
 
                     // rayleigh quotient residual bound.
                     if (a2 < cnst1) {
-                        s = gam * (1 - Math.sqrt(a2)) / (1 + a2);
+                        lowerSpectra = gam * (1 - Math.sqrt(a2)) / (1 + a2);
                     }
                     tau = s;
 
