diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/84/mutant-1/ori-MultiDirectional.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/84/mutant-1/fixed-patched-MultiDirectional.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/84/mutant-1/ori-MultiDirectional.java	2021-08-23 09:23:49.148994066 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/84/mutant-1/fixed-patched-MultiDirectional.java	2021-08-23 09:23:49.148994066 +0800
@@ -87,7 +87,7 @@
 
             // compute the contracted simplex
             final RealPointValuePair contracted = evaluateNewSimplex(original, gamma, comparator);
-            if (comparator.compare(contracted, best) < 0) {
+            if (comparator.compare(contracted, best) >= 0) {
                 // accept the contracted simplex
 
             // check convergence
