diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-93/ori-EigenDecompositionImpl.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-93/man-patched-EigenDecompositionImpl.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-93/ori-EigenDecompositionImpl.java	2021-08-23 09:23:49.140993955 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-93/man-patched-EigenDecompositionImpl.java	2021-08-23 09:23:49.140993955 +0800
@@ -1429,7 +1429,7 @@
      */
     private void computeShiftIncrement(final int start, final int end, final int deflated) {
 
-        final double cnst1 = 0.563;
+        double cnst1 = 0.563;
         final double cnst2 = 1.010;
         final double cnst3 = 1.05;
 
@@ -1477,7 +1477,7 @@
                     int np;
                     if (dMin == dN) {
                         gam = dN;
-                        a2 = 0.0;
+            cnst1 = 0.0D;
                         if (work[nn - 5]  >  work[nn - 7]) {
                             return;
                         }
