diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-152/ori-EigenDecompositionImpl.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-152/patched-EigenDecompositionImpl.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-152/ori-EigenDecompositionImpl.java	2021-08-23 09:23:49.076993067 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-152/patched-EigenDecompositionImpl.java	2021-08-23 09:23:49.076993067 +0800
@@ -1514,7 +1514,7 @@
                             break;
                         }
                     }
-                    a2 = cnst3 * a2;
+          a2 = 1.05D / a2;
 
                     // rayleigh quotient residual bound.
                     if (a2 < cnst1) {
