diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-54/ori-EigenDecompositionImpl.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-54/patched-EigenDecompositionImpl.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-54/ori-EigenDecompositionImpl.java	2021-08-23 09:23:49.108993511 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Math/81/mutant-54/patched-EigenDecompositionImpl.java	2021-08-23 09:23:49.108993511 +0800
@@ -1476,7 +1476,7 @@
                     double gam;
                     int np;
                     if (dMin == dN) {
-                        gam = dN;
+            gam = getDeterminant();
                         a2 = 0.0;
                         if (work[nn - 5]  >  work[nn - 7]) {
                             return;
