diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Mockito/8/mutant-3/ori-GenericMetadataSupport.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Mockito/8/mutant-3/man-patched-GenericMetadataSupport.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Mockito/8/mutant-3/ori-GenericMetadataSupport.java	2021-08-23 09:23:49.172994398 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Mockito/8/mutant-3/man-patched-GenericMetadataSupport.java	2021-08-23 09:23:49.172994398 +0800
@@ -373,9 +373,6 @@
         }
 
         private void readTypeVariables() {
-            for (Type type : typeVariable.getBounds()) {
-                registerTypeVariablesOn(type);
-            }
             registerTypeVariablesOn(getActualTypeArgumentFor(typeVariable));
         }
 
