diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/11/mutant-14/ori-DateTimeZoneBuilder.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/11/mutant-14/fixed-patched-DateTimeZoneBuilder.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/11/mutant-14/ori-DateTimeZoneBuilder.java	2021-08-23 09:23:49.180994509 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/11/mutant-14/fixed-patched-DateTimeZoneBuilder.java	2021-08-23 09:23:49.180994509 +0800
@@ -1141,7 +1141,7 @@
             if (iRules.size() == 2) {
                 Rule startRule = iRules.get(0);
                 Rule endRule = iRules.get(1);
-                if (startRule.getToYear() == Integer.MAX_VALUE &&
+                if (startRule.getFromYear() == Integer.MAX_VALUE &&
                     endRule.getToYear() == Integer.MAX_VALUE) {
 
                     // With exactly two infinitely recurring rules left, a
