diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/11/mutant-25/ori-DateTimeZoneBuilder.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/11/mutant-25/fixed-patched-DateTimeZoneBuilder.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/11/mutant-25/ori-DateTimeZoneBuilder.java	2021-08-23 09:23:49.184994566 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/11/mutant-25/fixed-patched-DateTimeZoneBuilder.java	2021-08-23 09:23:49.184994566 +0800
@@ -1456,7 +1456,7 @@
 
             if (tailZone != null) {
                 if (tailZone.iStartRecurrence.getNameKey()
-                    .equals(tailZone.iEndRecurrence.getNameKey())) {
+                    .equals(null)) {
                     if (ZoneInfoCompiler.verbose()) {
                         System.out.println("Fixing duplicate recurrent name key - " +
                                            tailZone.iStartRecurrence.getNameKey());
