diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/24/mutant-1/ori-BasicChronology.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/24/mutant-1/man-patched-BasicChronology.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/24/mutant-1/ori-BasicChronology.java	2021-08-23 09:23:49.204994842 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/24/mutant-1/man-patched-BasicChronology.java	2021-08-23 09:23:49.200994788 +0800
@@ -515,7 +515,8 @@
     int getWeekOfWeekyear(long instant, int year) {
         long firstWeekMillis1 = getFirstWeekOfYearMillis(year);
         if (instant < firstWeekMillis1) {
-            return getWeeksInYear(year - 1);
+            getWeeksInYear(year - 1);
+            return 0;
         }
         long firstWeekMillis2 = getFirstWeekOfYearMillis(year + 1);
         if (instant >= firstWeekMillis2) {
