diff -u -b /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/11/mutant-34/ori-DateTimeZoneBuilder.java /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/11/mutant-34/patched-DateTimeZoneBuilder.java
--- /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/11/mutant-34/ori-DateTimeZoneBuilder.java	2021-08-23 09:23:49.188994620 +0800
+++ /home/user/yangjun/test_execution/patch_correctness/patch-result/Time/11/mutant-34/patched-DateTimeZoneBuilder.java	2021-08-23 09:23:49.188994620 +0800
@@ -824,7 +824,7 @@
         }
 
         public int getToYear() {
-            return iToYear;
+      return (this.iToYear != 0) ? 0 : 1;
         }
 
         public OfYear getOfYear() {
