diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Nopol/Math/20/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Nopol/Math/20/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Nopol/Math/20/buggy1.java	2021-08-27 19:18:33.178863932 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Nopol/Math/20/tool-patch1.java	2021-08-27 19:18:33.174863932 -0700
@@ -701,7 +701,9 @@
                     .scalarMultiply(ccov1); // rank one update
             // minor correction if hsig==false
             double oldFac = hsig ? 0 : ccov1 * cc * (2. - cc);
+            if (org.apache.commons.math3.optimization.direct.CMAESOptimizer.this.inputSigma!=null) {
             oldFac += 1. - ccov1 - ccovmu;
+            }
             if (isActiveCMA) {
                 // Adapt covariance matrix C active CMA
                 negccov = (1. - ccovmu) * 0.25 * mueff /
