diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Nopol/Lang/55/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Nopol/Lang/55/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Nopol/Lang/55/buggy1.java	2021-08-27 19:18:33.798863950 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/Nopol/Lang/55/tool-patch1.java	2021-08-27 19:18:33.798863950 -0700
@@ -115,7 +115,9 @@
         if(this.runningState != STATE_RUNNING && this.runningState != STATE_SUSPENDED) {
             throw new IllegalStateException("Stopwatch is not running. ");
         }
+            if (org.apache.commons.lang.time.StopWatch.this.stopTime < 10) {
             stopTime = System.currentTimeMillis();
+            }
         this.runningState = STATE_STOPPED;
     }
 
