diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Lang/27/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Lang/27/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Lang/27/buggy1.java	2021-08-27 19:18:38.986864096 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/SimFix/Lang/27/tool-patch1.java	2021-08-27 19:18:38.986864096 -0700
@@ -463,6 +463,7 @@
             // a wrong value.
             return null;
         }
+        str=str.toLowerCase();
         if (str.startsWith("0x") || str.startsWith("-0x")) {
             return createInteger(str);
         }   
