diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/18/mutant-0/buggy-DefaultKeyedValues.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/18/mutant-0/patched-DefaultKeyedValues.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/18/mutant-0/buggy-DefaultKeyedValues.java	2021-08-22 00:18:38.566190183 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/18/mutant-0/patched-DefaultKeyedValues.java	2021-08-22 00:18:38.566190183 -0700
@@ -315,10 +315,8 @@
     public void removeValue(int index) {
         this.keys.remove(index);
         this.values.remove(index);
-        if (index < this.keys.size()) {
         rebuildIndex();
         }
-    }
 
     /**
      * Removes a value from the collection.
@@ -332,7 +330,8 @@
     public void removeValue(Comparable key) {
         int index = getIndex(key);
         if (index < 0) {
-			return;
+            throw new UnknownKeyException("The key (" + key 
+                    + ") is not recognised.");
         }
         removeValue(index);
     }
diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/18/mutant-0/buggy-DefaultKeyedValues2D.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/18/mutant-0/patched-DefaultKeyedValues2D.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/18/mutant-0/buggy-DefaultKeyedValues2D.java	2021-08-22 00:18:38.566190183 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/18/mutant-0/patched-DefaultKeyedValues2D.java	2021-08-22 00:18:38.566190183 -0700
@@ -452,11 +452,20 @@
      * @see #removeRow(Comparable)
      */
     public void removeColumn(Comparable columnKey) {
+    	if (columnKey == null) {
+    		throw new IllegalArgumentException("Null 'columnKey' argument.");
+    	}
+    	if (!this.columnKeys.contains(columnKey)) {
+    		throw new UnknownKeyException("Unknown key: " + columnKey);
+    	}
         Iterator iterator = this.rows.iterator();
         while (iterator.hasNext()) {
             DefaultKeyedValues rowData = (DefaultKeyedValues) iterator.next();
+            int index = rowData.getIndex(columnKey);
+            if (index >= 0) {
                 rowData.removeValue(columnKey);
         }
+        }
         this.columnKeys.remove(columnKey);
     }
 
