diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/10/mutant-0/buggy-StandardToolTipTagFragmentGenerator.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/10/mutant-0/patched-StandardToolTipTagFragmentGenerator.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/10/mutant-0/buggy-StandardToolTipTagFragmentGenerator.java	2021-08-22 00:18:38.562190182 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/10/mutant-0/patched-StandardToolTipTagFragmentGenerator.java	2021-08-22 00:18:38.562190182 -0700
@@ -62,7 +62,7 @@
      * @return The formatted HTML area tag attribute(s).
      */
     public String generateToolTipFragment(String toolTipText) {
-        return " title=\"" + toolTipText
+        return " title=\"" + ImageMapUtilities.htmlEscape(toolTipText) 
             + "\" alt=\"\"";
     }
 
