diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/9/mutant-0/buggy-TimeSeries.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/9/mutant-0/patched-TimeSeries.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/9/mutant-0/buggy-TimeSeries.java	2021-08-22 00:18:38.574190183 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/9/mutant-0/patched-TimeSeries.java	2021-08-22 00:18:38.574190183 -0700
@@ -941,7 +941,7 @@
             endIndex = -(endIndex + 1); // this is first item AFTER end period
             endIndex = endIndex - 1;    // so this is last item BEFORE end
         }
-        if (endIndex < 0) {
+        if ((endIndex < 0)  || (endIndex < startIndex)) {
             emptyRange = true;
         }
         if (emptyRange) {
