diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/13/mutant-0/buggy-BorderArrangement.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/13/mutant-0/patched-BorderArrangement.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/13/mutant-0/buggy-BorderArrangement.java	2021-08-22 00:18:38.562190182 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/13/mutant-0/patched-BorderArrangement.java	2021-08-22 00:18:38.562190182 -0700
@@ -452,7 +452,7 @@
         h[3] = h[2];
         if (this.rightBlock != null) {
             RectangleConstraint c4 = new RectangleConstraint(0.0,
-                    new Range(0.0, constraint.getWidth() - w[2]),
+                    new Range(0.0, Math.max(constraint.getWidth() - w[2], 0.0)),
                     LengthConstraintType.RANGE, h[2], null,
                     LengthConstraintType.FIXED);
             Size2D size = this.rightBlock.arrange(g2, c4);
