diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/17/mutant-0/buggy-TimeSeries.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/17/mutant-0/patched-TimeSeries.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/17/mutant-0/buggy-TimeSeries.java	2021-08-22 00:18:38.566190183 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Chart/17/mutant-0/patched-TimeSeries.java	2021-08-22 00:18:38.566190183 -0700
@@ -854,7 +854,8 @@
      *         subclasses may differ.
      */
     public Object clone() throws CloneNotSupportedException {
-        Object clone = createCopy(0, getItemCount() - 1);
+        TimeSeries clone = (TimeSeries) super.clone();
+        clone.data = (List) ObjectUtilities.deepClone(this.data);
         return clone;
     }
 
