diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/81/mutant-0/buggy-IRFactory.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/81/mutant-0/patched-IRFactory.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/81/mutant-0/buggy-IRFactory.java	2021-08-22 00:18:38.622190184 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/81/mutant-0/patched-IRFactory.java	2021-08-22 00:18:38.622190184 -0700
@@ -514,6 +514,13 @@
       Name name = functionNode.getFunctionName();
       Boolean isUnnamedFunction = false;
       if (name == null) {
+        int functionType = functionNode.getFunctionType();
+        if (functionType != FunctionNode.FUNCTION_EXPRESSION) {
+          errorReporter.error(
+            "unnamed function statement",
+            sourceName,
+            functionNode.getLineno(), "", 0);
+        }
         name = new Name();
         name.setIdentifier("");
         isUnnamedFunction = true;
