diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/89/mutant-0/buggy-CollapseProperties.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/89/mutant-0/patched-CollapseProperties.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/89/mutant-0/buggy-CollapseProperties.java	2021-08-22 00:18:38.626190184 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/89/mutant-0/patched-CollapseProperties.java	2021-08-22 00:18:38.626190184 -0700
@@ -481,6 +481,9 @@
     Node greatGramps = gramps.getParent();
     Node greatGreatGramps = greatGramps.getParent();
 
+    if (rvalue != null && rvalue.getType() == Token.FUNCTION) {
+      checkForHosedThisReferences(rvalue, refName.docInfo, refName);
+    }
 
     // Create the new alias node.
     Node nameNode = NodeUtil.newName(
diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/89/mutant-0/buggy-GlobalNamespace.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/89/mutant-0/patched-GlobalNamespace.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/89/mutant-0/buggy-GlobalNamespace.java	2021-08-22 00:18:38.626190184 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/89/mutant-0/patched-GlobalNamespace.java	2021-08-22 00:18:38.626190184 -0700
@@ -917,7 +917,7 @@
       }
 
       // If this is aliased, then its properties can't be collapsed either.
-      if (type != Type.FUNCTION && aliasingGets > 0) {
+      if (aliasingGets > 0) {
         return false;
       }
 
