diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/53/mutant-0/buggy-InlineObjectLiterals.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/53/mutant-0/patched-InlineObjectLiterals.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/53/mutant-0/buggy-InlineObjectLiterals.java	2021-08-22 00:18:38.610190184 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/53/mutant-0/patched-InlineObjectLiterals.java	2021-08-22 00:18:38.610190184 -0700
@@ -328,6 +328,9 @@
       }
 
       Node replacement;
+      if (nodes.isEmpty()) {
+        replacement = new Node(Token.TRUE);
+      } else {
         // All assignments evaluate to true, so make sure that the
         // expr statement evaluates to true in case it matters.
         nodes.add(new Node(Token.TRUE));
@@ -347,6 +350,7 @@
         }
         cur.addChildToFront(nodes.get(i));
         cur.addChildToFront(nodes.get(i + 1));
+      }
 
       Node replace = ref.getParent();
       replacement.copyInformationFromForTree(replace);
