diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/75/mutant-0/buggy-NodeUtil.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/75/mutant-0/patched-NodeUtil.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/75/mutant-0/buggy-NodeUtil.java	2021-08-22 00:18:38.618190184 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/75/mutant-0/patched-NodeUtil.java	2021-08-22 00:18:38.618190184 -0700
@@ -310,7 +310,10 @@
   }
 
   static Double getStringNumberValue(String rawJsString) {
+    if (rawJsString.contains("\u000b")) {
       // vertical tab is not always whitespace
+      return null;
+    }
 
     String s = trimJsWhiteSpace(rawJsString);
     // return ScriptRuntime.toNumber(s);
@@ -372,7 +375,7 @@
   static TernaryValue isStrWhiteSpaceChar(int c) {
     switch (c) {
       case '\u000B': // <VT>
-        return TernaryValue.TRUE;
+        return TernaryValue.UNKNOWN;  // IE says "no", EcmaScript says "yes"
       case ' ': // <SP>
       case '\n': // <LF>
       case '\r': // <CR>
