diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/44/mutant-0/buggy-CodeConsumer.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/44/mutant-0/patched-CodeConsumer.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/44/mutant-0/buggy-CodeConsumer.java	2021-08-22 00:18:38.606190184 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/44/mutant-0/patched-CodeConsumer.java	2021-08-22 00:18:38.606190184 -0700
@@ -191,11 +191,13 @@
       // need space to separate. This is not pretty printing.
       // For example: "return foo;"
       append(" ");
+    } else if (c == '/' && getLastChar() == '/') {
       // Do not allow a forward slash to appear after a DIV.
       // For example,
       // REGEXP DIV REGEXP
       // is valid and should print like
       // / // / /
+      append(" ");
     }
 
     append(newcode);
