diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/9/mutant-0/buggy-ProcessCommonJSModules.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/9/mutant-0/patched-ProcessCommonJSModules.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/9/mutant-0/buggy-ProcessCommonJSModules.java	2021-08-22 00:18:38.626190184 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/9/mutant-0/patched-ProcessCommonJSModules.java	2021-08-22 00:18:38.626190184 -0700
@@ -116,6 +116,7 @@
   private String normalizeSourceName(String filename) {
     // The DOS command shell will normalize "/" to "\", so we have to
     // wrestle it back.
+    filename = filename.replace("\\", "/");
 
     if (filename.indexOf(filenamePrefix) == 0) {
       filename = filename.substring(filenamePrefix.length());
@@ -180,7 +181,7 @@
       Preconditions.checkArgument(scriptNodeCount == 1,
           "ProcessCommonJSModules supports only one invocation per " +
           "CompilerInput / script node");
-      String moduleName = guessCJSModuleName(normalizeSourceName(script.getSourceFileName()));
+      String moduleName = guessCJSModuleName(script.getSourceFileName());
       script.addChildToFront(IR.var(IR.name(moduleName), IR.objectlit())
           .copyInformationFromForTree(script));
       if (reportDependencies) {
