diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/100/mutant-0/buggy-CheckGlobalThis.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/100/mutant-0/patched-CheckGlobalThis.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/100/mutant-0/buggy-CheckGlobalThis.java	2021-08-22 00:18:38.574190183 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/100/mutant-0/patched-CheckGlobalThis.java	2021-08-22 00:18:38.574190183 -0700
@@ -96,6 +96,13 @@
       // var a = function() { }; // or
       // function a() {} // or
       // a.x = function() {};
+      int pType = parent.getType();
+      if (!(pType == Token.BLOCK ||
+            pType == Token.SCRIPT ||
+            pType == Token.NAME ||
+            pType == Token.ASSIGN)) {
+        return false;
+      }
     }
 
     if (parent != null && parent.getType() == Token.ASSIGN) {
@@ -143,7 +150,7 @@
     }
 
     // Also report a THIS with a property access.
-    return false;
+    return parent != null && NodeUtil.isGet(parent);
   }
 
   /**
