diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/39/mutant-0/buggy-PrototypeObjectType.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/39/mutant-0/patched-PrototypeObjectType.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/39/mutant-0/buggy-PrototypeObjectType.java	2021-08-22 00:18:38.606190184 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/39/mutant-0/patched-PrototypeObjectType.java	2021-08-22 00:18:38.606190184 -0700
@@ -377,10 +377,10 @@
 
         sb.append(property);
         sb.append(": ");
-        sb.append(getPropertyType(property).toString());
+        sb.append(getPropertyType(property).toStringHelper(forAnnotations));
 
         ++i;
-        if (i == MAX_PRETTY_PRINTED_PROPERTIES) {
+        if (!forAnnotations && i == MAX_PRETTY_PRINTED_PROPERTIES) {
           sb.append(", ...");
           break;
         }
@@ -391,7 +391,7 @@
       prettyPrint = true;
       return sb.toString();
     } else {
-      return "{...}";
+      return forAnnotations ? "?" : "{...}";
     }
   }
 
