diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/7/mutant-0/buggy-ChainableReverseAbstractInterpreter.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/7/mutant-0/patched-ChainableReverseAbstractInterpreter.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/7/mutant-0/buggy-ChainableReverseAbstractInterpreter.java	2021-08-22 00:18:38.618190184 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/7/mutant-0/patched-ChainableReverseAbstractInterpreter.java	2021-08-22 00:18:38.618190184 -0700
@@ -610,9 +610,13 @@
     public JSType caseObjectType(ObjectType type) {
       if (value.equals("function")) {
         JSType ctorType = getNativeType(U2U_CONSTRUCTOR_TYPE);
-        return resultEqualsValue && ctorType.isSubtype(type) ? ctorType : null;
+        if (resultEqualsValue) {
           // Objects are restricted to "Function", subtypes are left
+          return ctorType.getGreatestSubtype(type);
+        } else {
           // Only filter out subtypes of "function"
+          return type.isSubtype(ctorType) ? null : type;
+        }
       }
       return matchesExpectation("object") ? type : null;
     }
