diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/26/mutant-0/buggy-ProcessCommonJSModules.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/26/mutant-0/patched-ProcessCommonJSModules.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/26/mutant-0/buggy-ProcessCommonJSModules.java	2021-08-22 00:18:38.598190183 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/26/mutant-0/patched-ProcessCommonJSModules.java	2021-08-22 00:18:38.598190183 -0700
@@ -125,6 +125,7 @@
       AbstractPostOrderCallback {
 
     private int scriptNodeCount = 0;
+    private Set<String> modulesWithExports = Sets.newHashSet();
 
     @Override
     public void visit(NodeTraversal t, Node n, Node parent) {
@@ -204,6 +205,9 @@
      */
     private void emitOptionalModuleExportsOverride(Node script,
         String moduleName) {
+      if (!modulesWithExports.contains(moduleName)) {
+        return;
+      }
 
       Node moduleExportsProp = IR.getprop(IR.name(moduleName),
           IR.string("module$exports"));
@@ -225,6 +229,7 @@
       Node exports = prop.getChildAtIndex(1);
       exports.putProp(Node.ORIGINALNAME_PROP, "exports");
       exports.setString("module$exports");
+      modulesWithExports.add(moduleName);
     }
 
     /**
