diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/46/mutant-0/buggy-RecordType.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/46/mutant-0/patched-RecordType.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/46/mutant-0/buggy-RecordType.java	2021-08-22 00:18:38.606190184 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/46/mutant-0/patched-RecordType.java	2021-08-22 00:18:38.606190184 -0700
@@ -137,22 +137,6 @@
         propertyNode);
   }
 
-  @Override
-  public JSType getLeastSupertype(JSType that) {
-    if (!that.isRecordType()) {
-      return super.getLeastSupertype(that);
-    }
-    RecordTypeBuilder builder = new RecordTypeBuilder(registry);
-    for (String property : properties.keySet()) {
-      if (that.toMaybeRecordType().hasProperty(property) &&
-          that.toMaybeRecordType().getPropertyType(property).isEquivalentTo(
-              getPropertyType(property))) {
-        builder.addProperty(property, getPropertyType(property),
-            getPropertyNode(property));
-      }
-    }
-    return builder.build();
-  }
   JSType getGreatestSubtypeHelper(JSType that) {
     if (that.isRecordType()) {
       RecordType thatRecord = that.toMaybeRecordType();
