diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/133/mutant-0/buggy-JsDocInfoParser.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/133/mutant-0/patched-JsDocInfoParser.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/133/mutant-0/buggy-JsDocInfoParser.java	2021-08-22 00:18:38.594190183 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/133/mutant-0/patched-JsDocInfoParser.java	2021-08-22 00:18:38.594190183 -0700
@@ -2398,6 +2398,7 @@
    */
   private String getRemainingJSDocLine() {
     String result = stream.getRemainingJSDocLine();
+    unreadToken = NO_UNREAD_TOKEN;
     return result;
   }
 
