diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/68/mutant-0/buggy-JsDocInfoParser.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/68/mutant-0/patched-JsDocInfoParser.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/68/mutant-0/buggy-JsDocInfoParser.java	2021-08-22 00:18:38.618190184 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/68/mutant-0/patched-JsDocInfoParser.java	2021-08-22 00:18:38.618190184 -0700
@@ -865,9 +865,9 @@
                         }
                         break;
                     }
+                  }
 
                   token = eatTokensUntilEOL();
-                  }
                   continue retry;
               }
             }
@@ -1706,6 +1706,7 @@
       }
     }
 
+    restoreLookAhead(token);
     return reportGenericTypeSyntaxWarning();
   }
 
@@ -1758,6 +1759,7 @@
     // NOTE(nicksantos): We're not implementing generics at the moment, so
     // just throw out TypeParameters.
     if (token != JsDocToken.LP) {
+      restoreLookAhead(token);
       return reportTypeSyntaxWarning("msg.jsdoc.missing.lp");
     }
 
