diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/6/mutant-0/buggy-TypeValidator.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/6/mutant-0/patched-TypeValidator.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/6/mutant-0/buggy-TypeValidator.java	2021-08-22 00:18:38.614190184 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/6/mutant-0/patched-TypeValidator.java	2021-08-22 00:18:38.614190184 -0700
@@ -363,9 +363,6 @@
       JSType leftType, Node owner, String propName) {
     // The NoType check is a hack to make typedefs work OK.
     if (!leftType.isNoType() && !rightType.canAssignTo(leftType)) {
-      if ((leftType.isConstructor() || leftType.isEnumType()) && (rightType.isConstructor() || rightType.isEnumType())) {
-        registerMismatch(rightType, leftType, null);
-      } else {
       // Do not type-check interface methods, because we expect that
       // they will have dummy implementations that do not match the type
       // annotations.
@@ -382,7 +379,6 @@
           "assignment to property " + propName + " of " +
           getReadableJSTypeName(owner, true),
           rightType, leftType);
-      }
       return false;
     }
     return true;
@@ -402,11 +398,7 @@
   boolean expectCanAssignTo(NodeTraversal t, Node n, JSType rightType,
       JSType leftType, String msg) {
     if (!rightType.canAssignTo(leftType)) {
-      if ((leftType.isConstructor() || leftType.isEnumType()) && (rightType.isConstructor() || rightType.isEnumType())) {
-        registerMismatch(rightType, leftType, null);
-      } else {
       mismatch(t, n, msg, rightType, leftType);
-      }
       return false;
     }
     return true;
