diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/14/mutant-0/buggy-ControlFlowAnalysis.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/14/mutant-0/patched-ControlFlowAnalysis.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/14/mutant-0/buggy-ControlFlowAnalysis.java	2021-08-22 00:18:38.594190183 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Closure/14/mutant-0/patched-ControlFlowAnalysis.java	2021-08-22 00:18:38.594190183 -0700
@@ -764,7 +764,7 @@
         } else if (parent.getLastChild() == node){
           if (cfa != null) {
             for (Node finallyNode : cfa.finallyMap.get(parent)) {
-              cfa.createEdge(fromNode, Branch.UNCOND, finallyNode);
+              cfa.createEdge(fromNode, Branch.ON_EX, finallyNode);
             }
           }
           return computeFollowNode(fromNode, parent, cfa);
