diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/15/mutant-0/buggy-TypeUtils.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/15/mutant-0/patched-TypeUtils.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/15/mutant-0/buggy-TypeUtils.java	2021-08-22 00:18:38.634190184 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/15/mutant-0/patched-TypeUtils.java	2021-08-22 00:18:38.634190184 -0700
@@ -216,9 +216,9 @@
                 toClass, typeVarAssigns);
 
         // now to check each type argument
-        for (Map.Entry<TypeVariable<?>, Type> entry : toTypeVarAssigns.entrySet()) {
-            Type toTypeArg = entry.getValue();
-            Type fromTypeArg = fromTypeVarAssigns.get(entry.getKey());
+        for (TypeVariable<?> var : toTypeVarAssigns.keySet()) {
+            Type toTypeArg = unrollVariableAssignments(var, toTypeVarAssigns);
+            Type fromTypeArg = unrollVariableAssignments(var, fromTypeVarAssigns);
 
             // parameters must either be absent from the subject type, within
             // the bounds of the wildcard type, or be an exact match to the
@@ -672,7 +672,7 @@
                 : new HashMap<TypeVariable<?>, Type>(subtypeVarAssigns);
 
         // has target class been reached?
-        if (cls.getTypeParameters().length > 0 || toClass.equals(cls)) {
+        if (toClass.equals(cls)) {
             return typeVarAssigns;
         }
 
