diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/36/mutant-0/buggy-NumberUtils.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/36/mutant-0/patched-NumberUtils.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/36/mutant-0/buggy-NumberUtils.java	2021-08-22 00:18:38.646190185 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/36/mutant-0/patched-NumberUtils.java	2021-08-22 00:18:38.646190185 -0700
@@ -488,7 +488,7 @@
             }
             dec = null;
         }
-        if (!Character.isDigit(lastChar)) {
+        if (!Character.isDigit(lastChar) && lastChar != '.') {
             if (expPos > -1 && expPos < str.length() - 1) {
                 exp = str.substring(expPos + 1, str.length() - 1);
             } else {
@@ -1385,6 +1385,14 @@
                 // can't have an E at the last byte
                 return false;
             }
+            if (chars[i] == '.') {
+                if (hasDecPoint || hasExp) {
+                    // two decimal points or dec in exponent
+                    return false;
+                }
+                // single trailing decimal point after non-exponent is ok
+                return foundDigit;
+            }
             if (!allowSigns
                 && (chars[i] == 'd'
                     || chars[i] == 'D'
