diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/14/mutant-0/buggy-StringUtils.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/14/mutant-0/patched-StringUtils.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/14/mutant-0/buggy-StringUtils.java	2021-08-22 00:18:38.634190184 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/14/mutant-0/patched-StringUtils.java	2021-08-22 00:18:38.634190184 -0700
@@ -785,8 +785,11 @@
         if (cs1 == null || cs2 == null) {
             return false;
         }
+        if (cs1 instanceof String && cs2 instanceof String) {
             return cs1.equals(cs2);
     }
+        return CharSequenceUtils.regionMatches(cs1, false, 0, cs2, 0, Math.max(cs1.length(), cs2.length()));
+    }
 
     /**
      * <p>Compares two CharSequences, returning {@code true} if they represent
