diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/48/mutant-0/buggy-EqualsBuilder.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/48/mutant-0/patched-EqualsBuilder.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/48/mutant-0/buggy-EqualsBuilder.java	2021-08-22 00:18:38.654190185 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/48/mutant-0/patched-EqualsBuilder.java	2021-08-22 00:18:38.654190185 -0700
@@ -377,8 +377,12 @@
         }
         Class lhsClass = lhs.getClass();
         if (!lhsClass.isArray()) {
+            if (lhs instanceof java.math.BigDecimal) {
+                isEquals = (((java.math.BigDecimal)lhs).compareTo(rhs) == 0);
+            } else {
                 // The simple case, not an array, just test the element
                 isEquals = lhs.equals(rhs);
+            }
         } else if (lhs.getClass() != rhs.getClass()) {
             // Here when we compare different dimensions, for example: a boolean[][] to a boolean[] 
             this.setEquals(false);
