diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/3/mutant-0/buggy-NumberUtils.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/3/mutant-0/patched-NumberUtils.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/3/mutant-0/buggy-NumberUtils.java	2021-08-22 00:18:38.642190185 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/3/mutant-0/patched-NumberUtils.java	2021-08-22 00:18:38.642190185 -0700
@@ -590,18 +590,22 @@
         //Must be a Float, Double, BigDecimal
         final boolean allZeros = isAllZeros(mant) && isAllZeros(exp);
         try {
+            if(numDecimals <= 7){// If number has 7 or fewer digits past the decimal point then make it a float
                 final Float f = createFloat(str);
                 if (!(f.isInfinite() || (f.floatValue() == 0.0F && !allZeros))) {
                     return f;
                 }
+            }
         } catch (final NumberFormatException nfe) { // NOPMD
             // ignore the bad number
         }
         try {
+            if(numDecimals <= 16){// If number has between 8 and 16 digits past the decimal point then make it a double
                 final Double d = createDouble(str);
                 if (!(d.isInfinite() || (d.doubleValue() == 0.0D && !allZeros))) {
                     return d;
                 }
+            }
         } catch (final NumberFormatException nfe) { // NOPMD
             // ignore the bad number
         }
