diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/22/mutant-0/buggy-Fraction.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/22/mutant-0/patched-Fraction.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/22/mutant-0/buggy-Fraction.java	2021-08-22 00:18:38.638190185 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/22/mutant-0/patched-Fraction.java	2021-08-22 00:18:38.638190185 -0700
@@ -580,8 +580,14 @@
      */
     private static int greatestCommonDivisor(int u, int v) {
         // From Commons Math:
+        if ((u == 0) || (v == 0)) {
+            if ((u == Integer.MIN_VALUE) || (v == Integer.MIN_VALUE)) {
+                throw new ArithmeticException("overflow: gcd is 2^31");
+            }
+            return Math.abs(u) + Math.abs(v);
+        }
         //if either operand is abs 1, return 1:
-        if (Math.abs(u) <= 1 || Math.abs(v) <= 1) {
+        if (Math.abs(u) == 1 || Math.abs(v) == 1) {
             return 1;
         }
         // keep u and v negative, as negative integers range down to
