diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/49/mutant-0/buggy-Fraction.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/49/mutant-0/patched-Fraction.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/49/mutant-0/buggy-Fraction.java	2021-08-22 00:18:38.654190185 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/49/mutant-0/patched-Fraction.java	2021-08-22 00:18:38.654190185 -0700
@@ -463,6 +463,9 @@
      * @return a new reduced fraction instance, or this if no simplification possible
      */
     public Fraction reduce() {
+        if (numerator == 0) {
+            return equals(ZERO) ? this : ZERO;
+        }
         int gcd = greatestCommonDivisor(Math.abs(numerator), denominator);
         if (gcd == 1) {
             return this;
