diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/9/mutant-0/buggy-FastDateParser.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/9/mutant-0/patched-FastDateParser.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/9/mutant-0/buggy-FastDateParser.java	2021-08-22 00:18:38.658190185 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/9/mutant-0/patched-FastDateParser.java	2021-08-22 00:18:38.658190185 -0700
@@ -141,6 +141,9 @@
             currentFormatField= nextFormatField;
             currentStrategy= nextStrategy;
         }
+        if (patternMatcher.regionStart() != patternMatcher.regionEnd()) {
+            throw new IllegalArgumentException("Failed to parse \""+pattern+"\" ; gave up at index "+patternMatcher.regionStart());
+        }
         if(currentStrategy.addRegex(this, regex)) {
             collector.add(currentStrategy);
         }
