diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/39/mutant-0/buggy-StringUtils.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/39/mutant-0/patched-StringUtils.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/39/mutant-0/buggy-StringUtils.java	2021-08-22 00:18:38.646190185 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/39/mutant-0/patched-StringUtils.java	2021-08-22 00:18:38.646190185 -0700
@@ -3673,6 +3673,9 @@
 
         // count the replacement text elements that are larger than their corresponding text being replaced
         for (int i = 0; i < searchList.length; i++) {
+            if (searchList[i] == null || replacementList[i] == null) {
+                continue;
+            }
             int greater = replacementList[i].length() - searchList[i].length();
             if (greater > 0) {
                 increase += 3 * greater; // assume 3 matches
