diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/23/mutant-0/buggy-ExtendedMessageFormat.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/23/mutant-0/patched-ExtendedMessageFormat.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/23/mutant-0/buggy-ExtendedMessageFormat.java	2021-08-22 00:18:38.638190185 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Lang/23/mutant-0/patched-ExtendedMessageFormat.java	2021-08-22 00:18:38.638190185 -0700
@@ -70,6 +70,7 @@
  */
 public class ExtendedMessageFormat extends MessageFormat {
     private static final long serialVersionUID = -2362048321261811743L;
+    private static final int HASH_SEED = 31;
 
     private static final String DUMMY_PATTERN = "";
     private static final String ESCAPED_QUOTE = "''";
@@ -260,12 +261,42 @@
      * @param obj the object to compare to
      * @return true if this object equals the other, otherwise false
      */
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == this) {
+            return true;
+        }
+        if (obj == null) {
+            return false;
+        }
+        if (!super.equals(obj)) {
+            return false;
+        }
+        if (ObjectUtils.notEqual(getClass(), obj.getClass())) {
+          return false;
+        }
+        ExtendedMessageFormat rhs = (ExtendedMessageFormat)obj;
+        if (ObjectUtils.notEqual(toPattern, rhs.toPattern)) {
+            return false;
+        }
+        if (ObjectUtils.notEqual(registry, rhs.registry)) {
+            return false;
+        }
+        return true;
+    }
 
     /**
      * Return the hashcode.
      *
      * @return the hashcode
      */
+    @Override
+    public int hashCode() {
+        int result = super.hashCode();
+        result = HASH_SEED * result + ObjectUtils.hashCode(registry);
+        result = HASH_SEED * result + ObjectUtils.hashCode(toPattern);
+        return result;
+    }
 
     /**
      * Get a custom format from a format description.
