diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/39/mutant-0/buggy-EmbeddedRungeKuttaIntegrator.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/39/mutant-0/patched-EmbeddedRungeKuttaIntegrator.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/39/mutant-0/buggy-EmbeddedRungeKuttaIntegrator.java	2021-08-22 00:18:38.674190186 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/39/mutant-0/patched-EmbeddedRungeKuttaIntegrator.java	2021-08-22 00:18:38.674190186 -0700
@@ -247,6 +247,15 @@
         }
 
         stepSize = hNew;
+        if (forward) {
+            if (stepStart + stepSize >= t) {
+                stepSize = t - stepStart;
+            }
+        } else {
+            if (stepStart + stepSize <= t) {
+                stepSize = t - stepStart;
+            }
+        }
 
         // next stages
         for (int k = 1; k < stages; ++k) {
