diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/17/mutant-0/buggy-Dfp.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/17/mutant-0/patched-Dfp.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/17/mutant-0/buggy-Dfp.java	2021-08-22 00:18:38.666190185 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/17/mutant-0/patched-Dfp.java	2021-08-22 00:18:38.666190185 -0700
@@ -1600,7 +1600,11 @@
      * @return product of this and x
      */
     public Dfp multiply(final int x) {
+        if (x >= 0 && x < RADIX) {
             return multiplyFast(x);
+        } else {
+            return multiply(newInstance(x));
+        }
     }
 
     /** Multiply this by a single digit 0&lt;=x&lt;radix.
