diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/82/mutant-0/buggy-SimplexSolver.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/82/mutant-0/patched-SimplexSolver.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/82/mutant-0/buggy-SimplexSolver.java	2021-08-22 00:18:38.694190186 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/82/mutant-0/patched-SimplexSolver.java	2021-08-22 00:18:38.694190186 -0700
@@ -79,7 +79,7 @@
         for (int i = tableau.getNumObjectiveFunctions(); i < tableau.getHeight(); i++) {
             final double rhs = tableau.getEntry(i, tableau.getWidth() - 1);
             final double entry = tableau.getEntry(i, col);
-            if (MathUtils.compareTo(entry, 0, epsilon) >= 0) {
+            if (MathUtils.compareTo(entry, 0, epsilon) > 0) {
                 final double ratio = rhs / entry;
                 if (ratio < minRatio) {
                     minRatio = ratio;
