diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/37/mutant-0/buggy-Complex.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/37/mutant-0/patched-Complex.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/37/mutant-0/buggy-Complex.java	2021-08-22 00:18:38.674190186 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/37/mutant-0/patched-Complex.java	2021-08-22 00:18:38.674190186 -0700
@@ -1015,9 +1015,15 @@
      * @since 1.2
      */
     public Complex tan() {
-        if (isNaN) {
+        if (isNaN || Double.isInfinite(real)) {
             return NaN;
         }
+        if (imaginary > 20.0) {
+            return createComplex(0.0, 1.0);
+        }
+        if (imaginary < -20.0) {
+            return createComplex(0.0, -1.0);
+        }
 
         double real2 = 2.0 * real;
         double imaginary2 = 2.0 * imaginary;
@@ -1060,9 +1066,15 @@
      * @since 1.2
      */
     public Complex tanh() {
-        if (isNaN) {
+        if (isNaN || Double.isInfinite(imaginary)) {
             return NaN;
         }
+        if (real > 20.0) {
+            return createComplex(1.0, 0.0);
+        }
+        if (real < -20.0) {
+            return createComplex(-1.0, 0.0);
+        }
         double real2 = 2.0 * real;
         double imaginary2 = 2.0 * imaginary;
         double d = FastMath.cosh(real2) + FastMath.cos(imaginary2);
