diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/86/mutant-0/buggy-CholeskyDecompositionImpl.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/86/mutant-0/patched-CholeskyDecompositionImpl.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/86/mutant-0/buggy-CholeskyDecompositionImpl.java	2021-08-22 00:18:38.694190186 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/86/mutant-0/patched-CholeskyDecompositionImpl.java	2021-08-22 00:18:38.694190186 -0700
@@ -111,9 +111,6 @@
 
             final double[] lI = lTData[i];
 
-            if (lTData[i][i] < absolutePositivityThreshold) {
-                throw new NotPositiveDefiniteMatrixException();
-            }
             // check off-diagonal elements (and reset them to 0)
             for (int j = i + 1; j < order; ++j) {
                 final double[] lJ = lTData[j];
@@ -134,6 +131,9 @@
             final double[] ltI = lTData[i];
 
             // check diagonal element
+            if (ltI[i] < absolutePositivityThreshold) {
+                throw new NotPositiveDefiniteMatrixException();
+            }
 
             ltI[i] = Math.sqrt(ltI[i]);
             final double inverse = 1.0 / ltI[i];
