diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/11/mutant-0/buggy-MultivariateNormalDistribution.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/11/mutant-0/patched-MultivariateNormalDistribution.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/11/mutant-0/buggy-MultivariateNormalDistribution.java	2021-08-22 00:18:38.662190185 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/11/mutant-0/patched-MultivariateNormalDistribution.java	2021-08-22 00:18:38.662190185 -0700
@@ -180,7 +180,7 @@
             throw new DimensionMismatchException(vals.length, dim);
         }
 
-        return FastMath.pow(2 * FastMath.PI, -dim / 2) *
+        return FastMath.pow(2 * FastMath.PI, -0.5 * dim) *
             FastMath.pow(covarianceMatrixDeterminant, -0.5) *
             getExponentTerm(vals);
     }
