diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/41/mutant-0/buggy-Variance.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/41/mutant-0/patched-Variance.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/41/mutant-0/buggy-Variance.java	2021-08-22 00:18:38.678190186 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/41/mutant-0/patched-Variance.java	2021-08-22 00:18:38.678190186 -0700
@@ -517,7 +517,7 @@
                 }
 
                 double sumWts = 0;
-                for (int i = 0; i < weights.length; i++) {
+                for (int i = begin; i < begin + length; i++) {
                     sumWts += weights[i];
                 }
 
