diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/90/mutant-0/buggy-Frequency.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/90/mutant-0/patched-Frequency.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/90/mutant-0/buggy-Frequency.java	2021-08-22 00:18:38.698190186 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/90/mutant-0/patched-Frequency.java	2021-08-22 00:18:38.698190186 -0700
@@ -107,6 +107,8 @@
      */
     @Deprecated
     public void addValue(Object v) {
+        addValue((Comparable<?>) v);
+    }
     
     /**
      * Adds 1 to the frequency count for v.
@@ -118,6 +120,7 @@
      * @param v the value to add.
      * @throws IllegalArgumentException if <code>v</code> is not comparable with previous entries
      */
+    public void addValue(Comparable<?>v){
         Object obj = v;
         if (v instanceof Integer) {
            obj = Long.valueOf(((Integer) v).longValue());
