diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/58/mutant-0/buggy-GaussianFitter.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/58/mutant-0/patched-GaussianFitter.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/58/mutant-0/buggy-GaussianFitter.java	2021-08-22 00:18:38.682190186 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/58/mutant-0/patched-GaussianFitter.java	2021-08-22 00:18:38.682190186 -0700
@@ -118,7 +118,7 @@
      */
     public double[] fit() {
         final double[] guess = (new ParameterGuesser(getObservations())).guess();
-        return fit(new Gaussian.Parametric(), guess);
+        return fit(guess);
     }
 
     /**
