diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/60/mutant-0/buggy-NormalDistributionImpl.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/60/mutant-0/patched-NormalDistributionImpl.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/60/mutant-0/buggy-NormalDistributionImpl.java	2021-08-22 00:18:38.686190186 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Math/60/mutant-0/patched-NormalDistributionImpl.java	2021-08-22 00:18:38.686190186 -0700
@@ -123,18 +123,11 @@
      */
     public double cumulativeProbability(double x) throws MathException {
         final double dev = x - mean;
-        try {
+        if (FastMath.abs(dev) > 40 * standardDeviation) { 
+            return dev < 0 ? 0.0d : 1.0d;
+        }
         return 0.5 * (1.0 + Erf.erf((dev) /
                     (standardDeviation * FastMath.sqrt(2.0))));
-        } catch (MaxIterationsExceededException ex) {
-            if (x < (mean - 20 * standardDeviation)) { // JDK 1.5 blows at 38
-                return 0;
-            } else if (x > (mean + 20 * standardDeviation)) {
-                return 1;
-            } else {
-                throw ex;
-            }
-        }
     }
 
     /**
