diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/27/mutant-0/buggy-MockUtil.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/27/mutant-0/patched-MockUtil.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/27/mutant-0/buggy-MockUtil.java	2021-08-22 00:18:38.706190186 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/27/mutant-0/patched-MockUtil.java	2021-08-22 00:18:38.706190186 -0700
@@ -61,8 +61,7 @@
 
     public <T> void resetMock(T mock) {
         MockHandlerInterface<T> oldMockHandler = getMockHandler(mock);
-        MockHandler<T> newMockHandler = new MockHandler<T>(oldMockHandler);
-        MethodInterceptorFilter newFilter = new MethodInterceptorFilter(newMockHandler, (MockSettingsImpl) org.mockito.Mockito.withSettings().defaultAnswer(org.mockito.Mockito.RETURNS_DEFAULTS));
+        MethodInterceptorFilter newFilter = newMethodInterceptorFilter(oldMockHandler.getMockSettings());
         ((Factory) mock).setCallback(0, newFilter);
     }
 
