diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/14/mutant-0/buggy-MockitoCore.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/14/mutant-0/patched-MockitoCore.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/14/mutant-0/buggy-MockitoCore.java	2021-08-22 00:18:38.702190186 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/14/mutant-0/patched-MockitoCore.java	2021-08-22 00:18:38.702190186 -0700
@@ -20,6 +20,7 @@
 import org.mockito.internal.stubbing.OngoingStubbingImpl;
 import org.mockito.internal.stubbing.StubberImpl;
 import org.mockito.internal.util.MockUtil;
+import org.mockito.internal.verification.MockAwareVerificationMode;
 import org.mockito.internal.verification.VerificationDataImpl;
 import org.mockito.internal.verification.VerificationModeFactory;
 import org.mockito.internal.verification.api.InOrderContext;
@@ -70,7 +71,7 @@
         } else if (!mockUtil.isMock(mock)) {
             reporter.notAMockPassedToVerify();
         }
-        mockingProgress.verificationStarted(mode);
+        mockingProgress.verificationStarted(new MockAwareVerificationMode(mock, mode));
         return mock;
     }
     
diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/14/mutant-0/buggy-MockHandler.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/14/mutant-0/patched-MockHandler.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/14/mutant-0/buggy-MockHandler.java	2021-08-22 00:18:38.702190186 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/14/mutant-0/patched-MockHandler.java	2021-08-22 00:18:38.702190186 -0700
@@ -17,6 +17,7 @@
 import org.mockito.internal.stubbing.OngoingStubbingImpl;
 import org.mockito.internal.stubbing.StubbedInvocationMatcher;
 import org.mockito.internal.stubbing.VoidMethodStubbableImpl;
+import org.mockito.internal.verification.MockAwareVerificationMode;
 import org.mockito.internal.verification.VerificationDataImpl;
 import org.mockito.stubbing.Answer;
 import org.mockito.stubbing.VoidMethodStubbable;
@@ -73,10 +74,12 @@
         if (verificationMode != null) {
             //We need to check if verification was started on the correct mock 
             // - see VerifyingWithAnExtraCallToADifferentMockTest
+            if (verificationMode instanceof MockAwareVerificationMode && ((MockAwareVerificationMode) verificationMode).getMock() == invocation.getMock()) {
                 VerificationDataImpl data = new VerificationDataImpl(invocationContainerImpl.getInvocations(), invocationMatcher);            
                 verificationMode.verify(data);
                 return null;
         }
+        }
         
         invocationContainerImpl.setInvocationForPotentialStubbing(invocationMatcher);
         OngoingStubbingImpl<T> ongoingStubbing = new OngoingStubbingImpl<T>(invocationContainerImpl);
