diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/26/mutant-0/buggy-Primitives.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/26/mutant-0/patched-Primitives.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/26/mutant-0/buggy-Primitives.java	2021-08-22 00:18:38.706190186 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Mockito/26/mutant-0/patched-Primitives.java	2021-08-22 00:18:38.706190186 -0700
@@ -63,6 +63,6 @@
         primitiveValues.put(int.class, 0);
         primitiveValues.put(long.class, 0L);
         primitiveValues.put(float.class, 0F);
-        primitiveValues.put(double.class, 0);
+        primitiveValues.put(double.class, 0D);
     }
 }
\ No newline at end of file
