diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/4/mutant-0/buggy-Partial.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/4/mutant-0/patched-Partial.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/4/mutant-0/buggy-Partial.java	2021-08-22 00:18:38.718190187 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/4/mutant-0/patched-Partial.java	2021-08-22 00:18:38.718190187 -0700
@@ -461,7 +461,7 @@
             System.arraycopy(iValues, i, newValues, i + 1, newValues.length - i - 1);
             // use public constructor to ensure full validation
             // this isn't overly efficient, but is safe
-            Partial newPartial = new Partial(iChronology, newTypes, newValues);
+            Partial newPartial = new Partial(newTypes, newValues, iChronology);
             iChronology.validate(newPartial, newValues);
             return newPartial;
         }
