diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/11/mutant-0/buggy-ZoneInfoCompiler.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/11/mutant-0/patched-ZoneInfoCompiler.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/11/mutant-0/buggy-ZoneInfoCompiler.java	2021-08-22 00:18:38.710190187 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/11/mutant-0/patched-ZoneInfoCompiler.java	2021-08-22 00:18:38.710190187 -0700
@@ -65,10 +65,11 @@
 
     static Chronology cLenientISO;
 
-    static ThreadLocal<Boolean> cVerbose = new ThreadLocal<Boolean>();
-    static {
-        cVerbose.set(Boolean.FALSE);
+    static ThreadLocal<Boolean> cVerbose = new ThreadLocal<Boolean>() {
+        protected Boolean initialValue() {
+            return Boolean.FALSE;
     }
+    };
 
     /**
      * Gets a flag indicating that verbose logging is required.
