diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/19/mutant-0/buggy-DateTimeZone.java /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/19/mutant-0/patched-DateTimeZone.java
--- /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/19/mutant-0/buggy-DateTimeZone.java	2021-08-22 00:18:38.714190187 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/developer_patches/Time/19/mutant-0/patched-DateTimeZone.java	2021-08-22 00:18:38.714190187 -0700
@@ -897,7 +897,7 @@
                     return offsetLocal;
                 }
             }
-        } else if (offsetLocal > 0) {
+        } else if (offsetLocal >= 0) {
             long prev = previousTransition(instantAdjusted);
             if (prev < instantAdjusted) {
                 int offsetPrev = getOffset(prev);
