diff -u -b /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/SketchFix/Math/82/2/buggy1.java /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/SketchFix/Math/82/2/tool-patch1.java
--- /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/SketchFix/Math/82/2/buggy1.java	2021-08-27 19:18:40.174864130 -0700
+++ /home/junyang/patch_correctness/gh/patch_correctness/ASE_Patches/Patches_others/Doverfitting/SketchFix/Math/82/2/tool-patch1.java	2021-08-27 19:18:40.174864130 -0700
@@ -81,7 +81,7 @@
             final double entry = tableau.getEntry(i, col);
             if (MathUtils.compareTo(entry, 0, epsilon) >= 0) {
                 final double ratio = rhs / entry;
-                if (ratio < minRatio) {
+                if (ratio < minRatio&&ratio>=0) {
                     minRatio = ratio;
                     minRatioPos = i; 
                 }
