diff -u -b /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/jMutRepair/Chart/25/buggy1.java /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/jMutRepair/Chart/25/tool-patch1.java
--- /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/jMutRepair/Chart/25/buggy1.java	2021-09-08 07:11:31.489222358 -0700
+++ /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_ICSE/Doverfitting/jMutRepair/Chart/25/tool-patch1.java	2021-09-08 07:11:31.489222358 -0700
@@ -107,7 +107,7 @@
         Number result = null;
         MeanAndStandardDeviation masd 
             = (MeanAndStandardDeviation) this.data.getObject(row, column);
-        if (masd != null) {
+        if (masd == null) {
             result = masd.getMean();
         }
         return result;
