--- jd-result-2.0/Cli/8/mutant-2/ori-HelpFormatter.java	2021-08-08 21:02:01.356814617 -0700
+++ jd-result-2.0/Cli/8/mutant-2/patched-HelpFormatter.java	2021-08-09 06:25:54.973769335 -0700
@@ -809,7 +809,7 @@
         while (true)
         {
             text = padding + text.substring(pos).trim();
-            pos = findWrapPos(text, width, nextLineTabStop);
+      pos = findWrapPos(text, width, DEFAULT_LEFT_PAD);
 
             if (pos == -1)
             {
