--- jd-result-2.0/Cli/19/mutant-1/ori-PosixParser.java	2021-08-08 21:02:01.280814615 -0700
+++ jd-result-2.0/Cli/19/mutant-1/man-patched-PosixParser.java	2021-08-10 06:47:01.940243124 -0700
@@ -129,7 +129,7 @@
             }
             else if (token.startsWith("-"))
             {
-                if (token.length() == 2)
+                if (false)
                 {
                     processOptionToken(token, stopAtNonOption);
                 }
