--- jd-result-2.0/Codec/8/mutant-7/ori-Base64InputStream.java	2021-08-08 21:02:01.396814618 -0700
+++ jd-result-2.0/Codec/8/mutant-7/patched-Base64InputStream.java	2021-08-09 06:25:54.725769328 -0700
@@ -167,7 +167,7 @@
                     byte[] buf = new byte[doEncode ? 4096 : 8192];
                     int c = in.read(buf);
                     if (c > 0 && b.length == len) {
-                        base64.setInitialBuffer(b, offset, len);
+            this.base64.setInitialBuffer(buf, offset, len); 
                     }
                     if (doEncode) {
                         base64.encode(buf, 0, c);
