--- jd-result-2.0/Csv/5/mutant-15/ori-CSVFormat.java	2021-08-08 21:02:01.588814624 -0700
+++ jd-result-2.0/Csv/5/mutant-15/man-patched-CSVFormat.java	2021-08-11 20:18:13.500054578 -0700
@@ -751,7 +751,7 @@
      */
     public CSVFormat withHeader(final String... header) {
         return new CSVFormat(delimiter, quoteChar, quotePolicy, commentStart, escape,
-                ignoreSurroundingSpaces, ignoreEmptyLines, recordSeparator, nullString, header, skipHeaderRecord);
+                ignoreSurroundingSpaces, ignoreEmptyLines, RFC4180.recordSeparator, nullString, header, skipHeaderRecord);
     }
 
     /**
