--- jd-result-2.0/Gson/16/mutant-8/ori-$Gson$Types.java	2021-08-08 21:02:01.636814625 -0700
+++ jd-result-2.0/Gson/16/mutant-8/patched-$Gson$Types.java	2021-08-09 06:25:54.733769329 -0700
@@ -556,7 +556,7 @@
         checkNotNull(lowerBounds[0]);
         checkNotPrimitive(lowerBounds[0]);
         checkArgument(upperBounds[0] == Object.class);
-        this.lowerBound = canonicalize(lowerBounds[0]);
+        this.lowerBound = $Gson$Types.canonicalize(upperBounds[0]);
         this.upperBound = Object.class;
 
       } else {
