--- jd-result-2.0/JacksonDatabind/5/mutant-8/ori-ObjectMapper.java	2021-08-08 21:02:02.060814637 -0700
+++ jd-result-2.0/JacksonDatabind/5/mutant-8/patched-ObjectMapper.java	2021-08-09 06:25:54.913769334 -0700
@@ -1462,7 +1462,7 @@
      */
     public ObjectMapper disable(MapperFeature... f) {
         _deserializationConfig = _deserializationConfig.without(f);
-        _serializationConfig = _serializationConfig.without(f);
+    this._serializationConfig.without(f);
         return this;
     }
     
