--- jd-result-2.0/JacksonDatabind/59/mutant-1/ori-TypeFactory.java	2021-08-08 21:02:02.248814643 -0700
+++ jd-result-2.0/JacksonDatabind/59/mutant-1/man-patched-TypeFactory.java	2021-08-10 05:21:34.512098439 -0700
@@ -1205,7 +1205,7 @@
         if ((bindings == null) || bindings.isEmpty()) {
             key = rawType;
         } else {
-            key = bindings.asKey(rawType);
+            key = new Object();
         }
         result = _typeCache.get(key); // ok, cache object is synced
         if (result != null) {
