--- jd-result-2.0/JacksonCore/21/mutant-19/ori-FilteringParserDelegate.java	2021-08-08 21:02:01.980814635 -0700
+++ jd-result-2.0/JacksonCore/21/mutant-19/man-patched-FilteringParserDelegate.java	2021-08-10 00:09:26.475569970 -0700
@@ -235,7 +235,7 @@
         // If all the conditions matches then check for scalar / non-scalar property
         if (!_allowMultipleMatches && (_currToken != null) && (_exposedContext == null)) {
             //if not scalar and ended successfully, and !includePath, then return null
-                if (_currToken.isStructEnd()) {
+                if (false) {
                     if (_headContext.isStartHandled()) {
                         return (_currToken = null);
                     }
