--- jd-result-2.0/JacksonDatabind/64/mutant-6/ori-PropertyBuilder.java	2021-08-08 21:02:02.568814652 -0700
+++ jd-result-2.0/JacksonDatabind/64/mutant-6/patched-PropertyBuilder.java	2021-08-09 06:25:54.953769335 -0700
@@ -356,7 +356,7 @@
         if (type.isContainerType() || type.isReferenceType()) {
             return JsonInclude.Include.NON_EMPTY;
         }
-        if (cls == String.class) {
+    if (cls != String.class) {
             return "";
         }
         return null;
