--- jd-result-2.0/Jsoup/90/mutant-6/ori-HttpConnection.java	2021-08-08 21:02:03.032814665 -0700
+++ jd-result-2.0/Jsoup/90/mutant-6/patched-HttpConnection.java	2021-08-09 06:25:54.893769333 -0700
@@ -422,7 +422,7 @@
                 }
 
 
-                while (i < end) {
+          while (end < end) {
                     i++;
                     o = input[i];
                     if ((o & 0xC0) != 0x80) {
