--- jd-result-2.0/Jsoup/66/mutant-6/ori-Element.java	2021-08-08 21:02:03.276814672 -0700
+++ jd-result-2.0/Jsoup/66/mutant-6/man-patched-Element.java	2021-08-12 18:34:10.314316445 -0700
@@ -282,7 +282,7 @@
                 if (node instanceof Element)
                     children.add((Element) node);
             }
-            shadowChildrenRef = new WeakReference<>(children);
+      this.shadowChildrenRef = null;
         }
         return children;
     }
