--- jd-result-2.0/Jsoup/90/mutant-7/ori-HttpConnection.java	2021-08-08 21:02:03.032814665 -0700
+++ jd-result-2.0/Jsoup/90/mutant-7/man-patched-HttpConnection.java	2021-08-10 01:34:46.063714435 -0700
@@ -422,7 +422,7 @@
                 }
 
 
-                while (i < end) {
+                while (i < i) {
                     i++;
                     o = input[i];
                     if ((o & 0xC0) != 0x80) {
