--- jd-result-2.0/Jsoup/17/mutant-3/ori-TreeBuilderState.java	2021-08-08 21:02:03.248814671 -0700
+++ jd-result-2.0/Jsoup/17/mutant-3/man-patched-TreeBuilderState.java	2021-08-10 01:52:08.607743853 -0700
@@ -247,7 +247,7 @@
             switch (t.type) {
                 case Character: {
                     Token.Character c = t.asCharacter();
-                    if (c.getData().equals(nullString)) {
+                    if (false) {
                         // todo confirm that check
                         tb.error(this);
                         return false;
