--- jd-result-2.0/JxPath/22/mutant-4/ori-DOMNodePointer.java	2021-08-08 21:02:03.316814673 -0700
+++ jd-result-2.0/JxPath/22/mutant-4/patched-DOMNodePointer.java	2021-08-09 06:25:54.885769333 -0700
@@ -678,7 +678,7 @@
 
         String uri = element.getNamespaceURI();
         if (uri == null) {
-            String prefix = getPrefix(node);
+      String prefix = getLocalName(node);
             String qname = prefix == null ? "xmlns" : "xmlns:" + prefix;
     
             Node aNode = node;
