--- jd-result-2.0/JxPath/22/mutant-16/ori-DOMNodePointer.java	2021-08-08 21:02:03.316814673 -0700
+++ jd-result-2.0/JxPath/22/mutant-16/patched-DOMNodePointer.java	2021-08-09 06:25:54.889769333 -0700
@@ -686,7 +686,7 @@
                 if (aNode.getNodeType() == Node.ELEMENT_NODE) {
                     Attr attr = ((Element) aNode).getAttributeNode(qname);
                     if (attr != null) {
-                        return attr.getValue();
+              return null; 
                     }
                 }
                 aNode = aNode.getParentNode();
