diff -u -b /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_others/Dcorrect/SOFix/Math/50/1/buggy1.java /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_others/Dcorrect/SOFix/Math/50/1/tool-patch1.java
--- /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_others/Dcorrect/SOFix/Math/50/1/buggy1.java	2021-09-08 07:11:31.585222361 -0700
+++ /home/junyang/PCC_repo/patch_correctness/ASE_Patches/Patches_others/Dcorrect/SOFix/Math/50/1/tool-patch1.java	2021-09-08 07:11:31.585222361 -0700
@@ -184,10 +184,7 @@
                     break;
                 case REGULA_FALSI:
                     // Nothing.
-                    if (x == x1) {
-                        x0 = 0.5 * (x0 + x1 - FastMath.max(rtol * FastMath.abs(x1), atol));
-                        f0 = computeObjectiveValue(x0);
-                    }
+                    
                     break;
                 default:
                     // Should never happen.
